
<HTML>
<HEAD>
<!-- This HTML file has been created by texi2html 1.39
     from formats.htmlinfo on 25 April 2016 -->

<TITLE>File Formats - Exp-Explain</TITLE>
</HEAD>
<BODY bgcolor="#ffffff">
<A HREF="formats_1.html"><IMG SRC="i/nav_first.gif" ALT="first"></A>  <A HREF="formats_19.html"><IMG SRC="i/nav_prev.gif" ALT="previous"></A>  <A HREF="formats_21.html"><IMG SRC="i/nav_next.gif" ALT="next"></A>  <A HREF="formats_26.html"><IMG SRC="i/nav_last.gif" ALT="last"></A>  <A HREF="formats_toc.html"><IMG SRC="i/nav_top.gif" ALT="contents"></A>
<HR>
<!-- NODE:Exp-Explain -->


<H3><A NAME="SEC42" HREF="formats_toc.html#SEC42">Explanation of Records</A></H3>
<P>
<A NAME="IDX42"></A>

</P>
<P>
<A NAME="IDX43"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
AC, ACcession line
<DT><CODE>Format</CODE>
<DD>
AC   string
<DT><CODE>Explanation</CODE>
<DD>
A unique identifier for the reading.
</DL>
<P>
<A NAME="IDX44"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
AP, Assembly Position
<DT><CODE>Format</CODE>
<DD>
AP   Name_of_anchor_reading sense offset tolerance
<DT><CODE>Explanation</CODE>
<DD>
For readings whose position has been mapped by an external program, these
records tell the "directed assembly" algorithm where to assemble the data.
Positions are defined as offsets from an "anchor reading" which is the name of
any reading already in the database, an orientation (sense, + or -), and a
tolerance. Readings are aligned at relative position offset + or - tolerance.
</DL>
<P>
<A NAME="IDX45"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
AQ, Average Quality of the reading.
<DT><CODE>Format</CODE>
<DD>
AQ   Numeric value in range 1 - 99.
<DT><CODE>Explanation</CODE>
<DD>
The average value of the "numerical estimate of base calling accuracy" as
calculated by program eba. The value is useful for monitoring data quality and
could also be used for deciding on an order of assembly - for example assemble
the highest quality readings first.
</DL>
<P>
<A NAME="IDX46"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
AV, Accuracy Values
<DT><CODE>Format</CODE>
<DD>
AV   q1 q2 q3 ... or a1,c1,g1,t1 a2,c2,g2,t2 ...
<DT><CODE>Explanation</CODE>
<DD>
The accuracy values lie in the range 1-99. Either 1 per base (eg 89 50 ...
or 4 per base (eg 0,89,5,2 50,3,7,10). <CITE>Bonfield,J.K and Staden,R.
The application of numerical estimates of base calling accuracy to DNA
sequencing projects. Nucleic Acids Res. 23 1406-1410, (1995)</CITE>.
</DL>
<P>
<A NAME="IDX47"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
BC, Base Calling software
</DL>
<P>
<A NAME="IDX48"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
CC, Comment line
<DT><CODE>Format</CODE>
<DD>
CC   string
<DT><CODE>Explanation</CODE>
<DD>
Any comments can be added on any number of lines.
</DL>
<P>
<A NAME="IDX49"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
CF, Cloning vector sequence File
<DT><CODE>Format</CODE>
<DD>
CF   string
<DT><CODE>Explanation</CODE>
<DD>
The name of the file containing the sequence of the cloning vector, to be used
by vector_clip (see section <A HREF="vector_clip_2.html#SEC2">Screening Against Vector Sequences</A>).

</DL>
<P>
<A NAME="IDX50"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
CH, Special CHemistry
<DT><CODE>Format</CODE>
<DD>
CH   number
<DT><CODE>Explanation</CODE>
<DD>
Used to flag readings as having been sequenced using a "special chemistry". The
number is a bit pattern with a bit for each chemistry type, thus allowing
combinations of chemistries to be listed. Currently bit 0 is used to
distinguish between dye-primer (0) and dye-terminator (1) chemistries. Bits 1
to 4 inclusive indicate the type of chemistry: unknown (0, 0000), ABI
Rhodamine (1, 0001), ABI dRhodamine (2, 0010), BigDye (3, 0011), Energy
Transfer (4, 0100) and LiCor (5, 0101). So for example a BigDye Terminator has 
bits 00111 set which is 7 in decimal.
</DL>
<P>
<A NAME="IDX51"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
CL, Cloning vector Left end
<DT><CODE>Format</CODE>
<DD>
CL   number
<DT><CODE>Explanation</CODE>
<DD>
The base position in the sequence that contains the last base in the cloning
vector. Currently gap4 only uses the CS line.
</DL>
<P>
<A NAME="IDX52"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
CN, Clone Name
<DT><CODE>Format</CODE>
<DD>
CN   string
<DT><CODE>Explanation</CODE>
<DD>
The name of the segment of DNA that the reading has been
derived from. Typically the name of a physical map clone. 
</DL>
<P>
<A NAME="IDX53"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
CR, Cloning vector Right end
<DT><CODE>Format</CODE>
<DD>
CR   number
<DT><CODE>Explanation</CODE>
<DD>
The base position in the sequence that contains the first base in the cloning
vector. Currently gap4 only uses the CS line.
</DL>
<P>
<A NAME="IDX54"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
CS, Cloning vector Sequence present in sequence
<DT><CODE>Format</CODE>
<DD>
CS   range
<DT><CODE>Explanation</CODE>
<DD>
Regions of sequence found by vector_clip 
(see section <A HREF="vector_clip_2.html#SEC2">Screening Against Vector Sequences</A>) to be cloning vector. Used in assembly to
exclude unwanted sequence.
</DL>
<P>
<A NAME="IDX55"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
CV, Cloning Vector type
<DT><CODE>Format</CODE>
<DD>
CV   string
<DT><CODE>Explanation</CODE>
<DD>
The type of the cloning vector used.
</DL>
<P>
<A NAME="IDX56"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
DR, Direction of Read
<DT><CODE>Format</CODE>
<DD>
DR   direction
<DT><CODE>Explanation</CODE>
<DD>
Whether forward or reverse primers were used. Allows
mapping of forward and reverse reads off the same template. NOTE however
that we do not encourage the use of this method as the terms
direction, sense and strand can be confusing. Instead we encourage the
use of the PRimer line.
</DL>
<P>
<A NAME="IDX57"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
DT, DaTe of experiment
<DT><CODE>Format</CODE>
<DD>
DT   dd-mon-yyyy
<DT><CODE>Explanation</CODE>
<DD>
Any date information.
</DL>
<P>
<A NAME="IDX58"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
EN, Entry Name
<DT><CODE>Format</CODE>
<DD>
EN   string
<DT><CODE>Explanation</CODE>
<DD>
The name given to the reading
</DL>
<P>
<A NAME="IDX59"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
EX, EXperimental notes
<DT><CODE>Format</CODE>
<DD>
EX   string
<DT><CODE>Explanation</CODE>
<DD>
Another type of comment line for additional information.
</DL>
<P>
<A NAME="IDX60"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
FM, sequencing vector Fragmentation Method
<DT><CODE>Format</CODE>
<DD>
FM   string
<DT><CODE>Explanation</CODE>
<DD>
Fragmentation method used to create sequencing library.
</DL>
<P>
<A NAME="IDX61"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
ID, IDentifier
<DT><CODE>Format</CODE>
<DD>
ID   string
<DT><CODE>Explanation</CODE>
<DD>
This is the name given to the reading inside the assembly database
and is equivalent to the ID line of an EMBL entry.
</DL>
<P>
<A NAME="IDX62"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
LE, Can be used to identify the location of materials
<DT><CODE>Format</CODE>
<DD>
LE   string
<DT><CODE>Explanation</CODE>
<DD>
Originally a micro titre dish well number. Used in
combination with LI.
</DL>
<P>
<A NAME="IDX63"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
LI, Can be used to identify the location of materials
<DT><CODE>Format</CODE>
<DD>
LI   string
<DT><CODE>Explanation</CODE>
<DD>
Originally a micro titre dish identifier. Used in
combination with LE.
</DL>
<P>
<A NAME="IDX64"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
LN, Local format trace file Name
<DT><CODE>Format</CODE>
<DD>
LN   string
<DT><CODE>Explanation</CODE>
<DD>
The name of the local format trace file. This information is passed
onto gap4, and allows for local formats to be used.
</DL>
<P>
<A NAME="IDX65"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
LT, Local format trace file Type
<DT><CODE>Format</CODE>
<DD>
LT   string
<DT><CODE>Explanation</CODE>
<DD>
The type of the local trace file type (usually SCF).
</DL>
<P>
<A NAME="IDX66"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
MC, MaChine on which sequencing experiment was run
<DT><CODE>Format</CODE>
<DD>
MC   string
<DT><CODE>Explanation</CODE>
<DD>
The lab's name for the sequencing machine used to create the data.
Used for logging the performance of individual machines.
</DL>
<P>
<A NAME="IDX67"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
MN, Machine generated trace file Name
<DT><CODE>Format</CODE>
<DD>
MN   string
<DT><CODE>Explanation</CODE>
<DD>
The name of the trace file generated by the sequencing machine MC.
</DL>
<P>
<A NAME="IDX68"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
MT, Machine generated trace file Type
<DT><CODE>Format</CODE>
<DD>
MT   string
<DT><CODE>Explanation</CODE>
<DD>
The type of machine generated trace file.
</DL>
<P>
<A NAME="IDX69"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
ON, Original base Numbers (positions)
<DT><CODE>Format</CODE>
<DD>
ON   (eg) 1..43 0 45..63 65..74 0 75..536
<DT><CODE>Explanation</CODE>
<DD>
The A..B notation means that values A to B inclusive, so this example reads
that bases 1 to 43 are unchanged, there is a change at 44, etc.
</DL>
<P>
<A NAME="IDX70"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
OP, OPerator
<DT><CODE>Format</CODE>
<DD>
OP   string
<DT><CODE>Explanation</CODE>
<DD>
Someone's name, possibly the person who ran the
sequencing machine. Useful, with expansion of the string field for
monitoring the performance of individuals!
</DL>
<P>
<A NAME="IDX71"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
PC,  Position in Contig
<DT><CODE>Format</CODE>
<DD>
PC    number
<DT><CODE>Explanation</CODE>
<DD>
For preassembled data, the position to put the left end of the reading.
</DL>
<P>
<A NAME="IDX72"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
PD,  Primer Data
<DT><CODE>Format</CODE>
<DD>
PD    sequence
<DT><CODE>Explanation</CODE>
<DD>
The primer sequence.
</DL>
<P>
<A NAME="IDX73"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
PN, Primer Name
<DT><CODE>Format</CODE>
<DD>
PN   string
<DT><CODE>Explanation</CODE>
<DD>
Name of primer used, using local naming convention. Could be a
universal primer. 
</DL>
<P>
<A NAME="IDX74"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
PR, PRimer type
<DT><CODE>Format</CODE>
<DD>
PR   number
<DT><CODE>Explanation</CODE>
<DD>
This record shows the direction of the reading and distinguishes between
primers from the ends of the insert and those that are internal. It is
important for the analysis of the relative orientations and positions of
readings on templates. When the positions of readings on templates are
analysed (see section <A HREF="gap5_45.html#SEC82">Find read pairs</A>) primer types
1,2,3 and 4 are represented using the symbols F,R,f and r respectively.

<DL COMPACT>

<DT><VAR>0</VAR>
<DD>
Unknown
<DT><VAR>1</VAR>
<DD>
Forward from beginning of insert
<DT><VAR>2</VAR>
<DD>
Reverse from end of insert
<DT><VAR>3</VAR>
<DD>
Custom forward i.e. a forward primer other than type 1.
<DT><VAR>4</VAR>
<DD>
Custom reverse i.e. a reverse primer other than type 2.
</DL>
</DL>
<P>
<A NAME="IDX75"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
PS, Processing Status
<DT><CODE>Format</CODE>
<DD>
PS   explanation
<DT><CODE>Explanation</CODE>
<DD>
Indication of processing status. 
</DL>
<P>
<A NAME="IDX76"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
QL, poor Quality sequence present at Left (5') end
<DT><CODE>Format</CODE>
<DD>
QL   position
<DT><CODE>Explanation</CODE>
<DD>
The sequence up to and including the base at the marked position are
considered to be of too poor quality to be used. 
It may overlap with other marked
sequences - CS, SL or SR. Used in assembly to exclude unwanted sequence.
</DL>
<P>
<A NAME="IDX77"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
QR, poor Quality sequence present at Right (3') end
<DT><CODE>Format</CODE>
<DD>
QR   position
<DT><CODE>Explanation</CODE>
<DD>
The sequence from and including the base at the marked position to the
end is considered to be of too poor quality to be used. It may overlap with
other marked sequences - CS, SL or SR. Used in assembly to exclude
unwanted sequence.
</DL>
<P>
<A NAME="IDX78"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
RS, Reference Sequence
<DT><CODE>Format</CODE>
<DD>
RS   string
<DT><CODE>Explanation</CODE>
<DD>
The name of a sequence, usually in EMBL format, used to define the target
sequence, base numbering 
and feature table data for a project. Used to define the numbering and
changes produced by mutations in individual sequence readings
(see section <A HREF="mutations_2.html#SEC2">Introduction to mutation detection</A>).
</DL>
<P>
<A NAME="IDX79"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SC, Sequencing vector Cloning site
<DT><CODE>Format</CODE>
<DD>
SC   position
<DT><CODE>Explanation</CODE>
<DD>
The cloning site of the sequence vector. Used by vector_clip 
(see section <A HREF="vector_clip_2.html#SEC2">Screening Against Vector Sequences</A>).
</DL>
<P>
<A NAME="IDX80"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SE, SEnse (ie whether complemented)
<DT><CODE>Format</CODE>
<DD>
SE   number
<DT><CODE>Explanation</CODE>
<DD>
For preassembled data, the sense of the reading (0 for forward, 1 for
reverse).
</DL>
<P>
<A NAME="IDX81"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SF, Sequencing vector sequence File
<DT><CODE>Format</CODE>
<DD>
SF   string
<DT><CODE>Explanation</CODE>
<DD>
The name of the file containing the sequence of the 
sequencing vector, to be used by vector_clip 
(see section <A HREF="vector_clip_2.html#SEC2">Screening Against Vector Sequences</A>).
</DL>
<P>
<A NAME="IDX82"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SI, Sequencing vector Insertion length
<DT><CODE>Format</CODE>
<DD>
SI   range
<DT><CODE>Explanation</CODE>
<DD>
Expected insertion length of sequence in sequencing
vector. Useful for selecting templates for further experiments.
</DL>
<P>
<A NAME="IDX83"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SL, Sequencing vector sequence present at Left (5') end
<DT><CODE>Format</CODE>
<DD>
SL   position
<DT><CODE>Explanation</CODE>
<DD>
The sequence up to and including the base at the marked 
position are considered to be sequencing vector. Written by vector_clip
(see section <A HREF="vector_clip_2.html#SEC2">Screening Against Vector Sequences</A>).
</DL>
<P>
<A NAME="IDX84"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SP, Sequencing vector Primer site (relative to cloning site)
<DT><CODE>Format</CODE>
<DD>
SP   position
<DT><CODE>Explanation</CODE>
<DD>
Location of the primer using to sequence relative to cloning site.
Used by vector_clip 
(see section <A HREF="vector_clip_2.html#SEC2">Screening Against Vector Sequences</A>).
</DL>
<P>
<A NAME="IDX85"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SQ, SeQuence
<DT><CODE>Format</CODE>
<DD>
SQ   \nsequence blocks...\n//\n
<DT><CODE>Explanation</CODE>
<DD>
Complete sequence, as determined by the sequencing machine. The sequence is
broken into blocks of 10 bases with 6 blocks per line separated by a space
(see the example below).
</DL>
<P>
<A NAME="IDX86"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SR, Sequencing vector sequence present at Right (3') end
<DT><CODE>Format</CODE>
<DD>
SR   position
<DT><CODE>Explanation</CODE>
<DD>
The sequence from and including the base at the marked 
position to the end are considered to be sequencing vector. Written by
vector_clip 
(see section <A HREF="vector_clip_2.html#SEC2">Screening Against Vector Sequences</A>).
</DL>
<P>
<A NAME="IDX87"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SS, Screening Sequence
<DT><CODE>Format</CODE>
<DD>
SS   string
<DT><CODE>Explanation</CODE>
<DD>
Note that in earlier versions of this documentation this field was explained
incorrectly. Due to this the field is not currently being used by any of our
programs. The original meaning was to specify a sequence to screen against.
Any number of SS lines could be present to denote any number of screening
sequences. In the future we may change the meaning of this field to be a
single SS line containing a file of filenames of screening sequences. If this
causes problems for people then we will choose a new line type, so please
inform us now. Also note that contrary to previous documentation, vector_clip does
not use this field (it uses the SF field instead).
</DL>
<P>
<A NAME="IDX88"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
ST, STrands
<DT><CODE>Format</CODE>
<DD>
ST   number
<DT><CODE>Explanation</CODE>
<DD>
Denotes whether this is a single or double stranded template. This
is useful for deducing suitable templates for later experiments.
</DL>
<P>
<A NAME="IDX89"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
SV, Sequencing Vector type
<DT><CODE>Format</CODE>
<DD>
SV   string
<DT><CODE>Explanation</CODE>
<DD>
Type of sequencing vector used. Can be used for choosing
templates for custom primer experiments.
</DL>
<P>
<A NAME="IDX90"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
TC, Tag to be placed on the Consensus.
<DT><CODE>Format</CODE>
<DD>
TC   TYPE S position..length
<DT><CODE>Explanation</CODE>
<DD>
These lines instruct gap4 to place tags on the consensus.
The format defines the tag type which is a 4 character identifier
and should start at column position 5), its strand  ( "+", "-" or
"=" which means both strands), its start position followed by the
position of its end. These two values are separated by "..". Following
lines starting TG with space characters up to column 10 are written
into the comment field of the tag. For example the next three lines
define a tag of type comment that is to be on both strands over the
range 100 to 110 and the comment field will contain "This comment
contains several lines".

<PRE>
TC   COMM = 100..110
TC        This comment contains
TC          several lines
</PRE>

</DL>
<P>
<A NAME="IDX91"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
TG, Tag to be placed on the reading.
<DT><CODE>Format</CODE>
<DD>
TG   TYPE S position..length
<DT><CODE>Explanation</CODE>
<DD>
These lines instruct gap4 to place tags on the reading.
See TC for further information.
</DL>
<P>
<A NAME="IDX92"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
TN, Template Name
<DT><CODE>Format</CODE>
<DD>
TN   string
<DT><CODE>Explanation</CODE>
<DD>
The name of the template used in the experiment.
</DL>
<P>
<A NAME="IDX93"></A>
<DL COMPACT>

<DT><CODE>Record</CODE>
<DD>
WT, Wild Type trace file
<DT><CODE>Format</CODE>
<DD>
WT   string
<DT><CODE>Explanation</CODE>
<DD>
The filename of the wild type trace file. Used for mutation studies.
</DL>

<HR>
<A HREF="formats_1.html"><IMG SRC="i/nav_first.gif" ALT="first"></A>  <A HREF="formats_19.html"><IMG SRC="i/nav_prev.gif" ALT="previous"></A>  <A HREF="formats_21.html"><IMG SRC="i/nav_next.gif" ALT="next"></A>  <A HREF="formats_26.html"><IMG SRC="i/nav_last.gif" ALT="last"></A>  <A HREF="formats_toc.html"><IMG SRC="i/nav_top.gif" ALT="contents"></A>
<hr>
<i>Last generated on 25 April 2016.</i>
<font size="-1"><br>
</font>
</BODY>
</HTML>
