
<HTML>
<HEAD>
<!-- This HTML file has been created by texi2html 1.39
     from gap4.htmlinfo on 25 April 2016 -->

<TITLE>Gap4 - Gap4-Introduction</TITLE>
</HEAD>
<BODY bgcolor="#ffffff">
<A HREF="gap4_1.html"><IMG SRC="i/nav_first.gif" ALT="first"></A>  <A HREF="gap4_2.html"><IMG SRC="i/nav_prev.gif" ALT="previous"></A>  <A HREF="gap4_4.html"><IMG SRC="i/nav_next.gif" ALT="next"></A>  <A HREF="gap4_182.html"><IMG SRC="i/nav_last.gif" ALT="last"></A>  <A HREF="gap4_toc.html"><IMG SRC="i/nav_top.gif" ALT="contents"></A>
<HR>
<!-- NODE:Gap4-Introduction -->


<H1><A NAME="SEC3" HREF="gap4_toc.html#SEC3">Introduction</A></H1>
<P>
<A NAME="IDX1"></A>

<UL>
<LI><A HREF="gap4_4.html#SEC4">Summary of the Files used and the Preprocessing Steps</A>
<LI><A HREF="gap4_5.html#SEC5">Summary of Gap4's Functions</A>
<LI><A HREF="gap4_6.html#SEC6">Introduction to the Gap4 User Interface</A>
<LI><A HREF="gap4_7.html#SEC7">Introduction to the Gap4 Contig Selector</A>
<LI><A HREF="gap4_8.html#SEC8">Introduction to the Gap4 Contig Comparator</A>
<LI><A HREF="gap4_9.html#SEC9">Introduction to the Gap4 Template Display</A>
<LI><A HREF="gap4_10.html#SEC10">Introduction to the Gap4 Consistency Display</A>
<LI><A HREF="gap4_11.html#SEC11">Introduction to the Gap4 Restriction Map</A>
<LI><A HREF="gap4_12.html#SEC12">Introduction to the Gap4 Stop Codon Map</A>
<LI><A HREF="gap4_13.html#SEC13">Introduction to the Gap4 Contig Editor</A>
<LI><A HREF="gap4_14.html#SEC14">Introduction to the Gap4 Contig Joining Editor</A>
</UL>

<P>
Gap4 
is a Genome Assembly Program.  
The program contains all the tools that would be expected from an assembly
program plus many unique features and a very easily used interface.
The original version was described in
<CITE>Bonfield,J.K., Smith,K.F. and
Staden,R. A new DNA sequence assembly program. Nucleic Acids Res. 24,
4992-4999 (1995)</CITE> 

</P>
<P>
Gap4 is very big and powerful. Everybody employs a subset of options and
has their favourite way of accessing and using them. Although there is a
lot of it, users are encouraged to go through the whole of the documentation
once, just to discover what is possible, and the way that best suits
their own work. At the very least, the whole of this introductory
chapter should be read, as in the long run, it will save time.

</P>
<P>
This chapter serves as a cross reference point, to give an overview of the
program and to introduce some of the important ideas which it uses. The
main topics that are introduced are listed in the current section. We
introduced the use of base call accuracy values for speeding up
sequencing projects 
(see section <A HREF="gap4_16.html#SEC23">The use of numerical estimates of base calling accuracy</A>).
The ability to annotate segments of readings and the consensus can be very
convenient 
(see section <A HREF="gap4_18.html#SEC25">Annotating and masking readings and contigs</A>).
Generally the 3' ends of readings from sequencing instruments are of too
low a quality to be used to create reliable consensus, but they can be
useful, for example, for finding joins between contigs
(see section <A HREF="gap4_17.html#SEC24">Use of the "hidden" poor quality data</A>).

</P>
<P>
One of the most powerful features of gap4 is its graphical user
interface which enables the data to be viewed and manipulated at several
levels of resolution. The displays which provide these different views are 
introduced, with several screenshots
(see section <A HREF="gap4_6.html#SEC6">Introduction to the gap4 User Interface</A>).

</P>
<P>
It is important to understand the different files used by our
sequence assembly software, and how the data is processed before it
reaches gap4
(see section <A HREF="gap4_4.html#SEC4">Summary of the Files used and the Preprocessing Steps</A>).

</P>
<P>
Note that gap4 is a very flexible program, and is designed so that it
can easily be configured to suit different purposes and ways of
working. For example it is easy to create a beginners 
version of gap4 which has
only a subset of functions. What is described in this manual is the full
version, and so is likely to contain some perhaps more esoteric options
that few people will need to use.
This introductory section also contains a
complete list of the options in the gap4 main menus
(see section <A HREF="gap4_15.html#SEC15">Gap4 Menus</A>).

</P>
<P>
In addition to sequence assembly, gap4 can be used for managing mutation
study data and for helping to discover and check for mutations
(see section <A HREF="mutations_2.html#SEC2">Introduction to Searching for Mutations</A>).

</P>
<P>
Two further useful facilities of gap4 are "Lists" and "Notes".
For many operations it is convenient to be able to process sets of data 
together - for example to
calculate a consensus sequence for a subset of the contigs. To
facilitate this gap4 uses lists
(see section <A HREF="gap5_69.html#SEC107">Lists Introduction</A>)
A `Note' 
(see section <A HREF="gap4_148.html#SEC220">Notes</A>)
is an arbitrary piece of text which can be attached to any
reading, any contig, or to the
database in general. 

</P>

<HR>
<A HREF="gap4_1.html"><IMG SRC="i/nav_first.gif" ALT="first"></A>  <A HREF="gap4_2.html"><IMG SRC="i/nav_prev.gif" ALT="previous"></A>  <A HREF="gap4_4.html"><IMG SRC="i/nav_next.gif" ALT="next"></A>  <A HREF="gap4_182.html"><IMG SRC="i/nav_last.gif" ALT="last"></A>  <A HREF="gap4_toc.html"><IMG SRC="i/nav_top.gif" ALT="contents"></A>
<hr>
<i>Last generated on 25 April 2016.</i>
<font size="-1"><br>
</font>
</BODY>
</HTML>
