
<HTML>
<HEAD>
<!-- This HTML file has been created by texi2html 1.39
     from gap5.htmlinfo on 25 April 2016 -->

<TITLE>Gap5 - Table of Contents</TITLE>
</HEAD>
<BODY bgcolor="#ffffff">
<a href="../index.html"><img src="i/nav_home.gif" alt="home"></a>
<a href="master_brief.html"><img src="i/nav_brief.gif" alt="brief"></a>
<a href="master_contents.html"><img src="i/nav_full.gif" alt="full"></a>
<hr size=4>
<H1>Gap5</H1>
<ADDRESS>James Bonfield, Wellcome Trust Sanger Institute</ADDRESS>
<P>
<UL>
<LI><A NAME="SEC1" HREF="gap5_1.html">Copyright</A>
<!-- NODE:Gap5_DB --><LI><A NAME="SEC2" HREF="gap5_2.html">Gap5 Databases</A>
<UL>
<!-- NODE:tg_index --><LI><A NAME="SEC3" HREF="gap5_2.html#SEC3">Creating databases</A>
<!-- NODE:OpenDatabase --><LI><A NAME="SEC4" HREF="gap5_2.html#SEC4">Opening/closing databases</A>
<!-- NODE:GapDB-Directories --><LI><A NAME="SEC5" HREF="gap5_2.html#SEC5">Changing directories</A>
<!-- NODE:CheckDatabase --><LI><A NAME="SEC6" HREF="gap5_2.html#SEC6">Check Database</A>
</UL>
<!-- NODE:Contig Selector and Comparator --><LI><A NAME="SEC7" HREF="gap5_3.html">Contig Selector / Comparator</A>
<UL>
<!-- NODE:Contig Selector --><LI><A NAME="SEC8" HREF="gap5_3.html#SEC8">Contig Selector</A>
<!-- NODE:Contig-Selector-Contigs --><LI><A NAME="SEC9" HREF="gap5_4.html">Selecting Contigs</A>
<!-- NODE:Contig-Selector-Order --><LI><A NAME="SEC10" HREF="gap5_5.html">Changing the Contig Order</A>
<!-- NODE:Contig-Selector-Menus --><LI><A NAME="SEC11" HREF="gap5_6.html">The Contig Selector Menus</A>
<!-- NODE:Contig Comparator --><LI><A NAME="SEC12" HREF="gap5_7.html">Contig Comparator</A>
<!-- NODE:Compar-Examining --><LI><A NAME="SEC13" HREF="gap5_8.html">Examining Results and Using Them to Select Commands</A>
<!-- NODE:Compar-AutoNavigation --><LI><A NAME="SEC14" HREF="gap5_9.html">Automatic Match Navigation</A>
</UL>
<!-- NODE:Template Display --><LI><A NAME="SEC15" HREF="gap5_10.html">Template Display</A>
<UL>
<!-- NODE:Template-Filter --><LI><A NAME="SEC16" HREF="gap5_10.html#SEC16">Filtering data</A>
<!-- NODE:Template-Template --><LI><A NAME="SEC17" HREF="gap5_10.html#SEC17">Template plot</A>
<UL>
<!-- NODE:Template-Y-Type --><LI><A NAME="SEC18" HREF="gap5_10.html#SEC18">Controlling The Y Layout.</A>
</UL>
<!-- NODE:Template-Depth --><LI><A NAME="SEC19" HREF="gap5_10.html#SEC19">Depth / Coverage Plot</A>
</UL>
<!-- NODE:Editor --><LI><A NAME="SEC20" HREF="gap5_11.html">Editing in Gap5</A>
<UL>
<!-- NODE:Editor-Movement --><LI><A NAME="SEC21" HREF="gap5_12.html">Moving the visible segment of the contig</A>
<!-- NODE:Editor-Names --><LI><A NAME="SEC22" HREF="gap5_13.html">Names</A>
<!-- NODE:Editor-Editing --><LI><A NAME="SEC23" HREF="gap5_14.html">Editing</A>
<UL>
<!-- NODE:Editor-Cursor --><LI><A NAME="SEC24" HREF="gap5_15.html">Moving the editing cursor</A>
<!-- NODE:Editor-Quality Values --><LI><A NAME="SEC25" HREF="gap5_16.html">Adjusting the Quality Values</A>
<!-- NODE:Editor-Positions --><LI><A NAME="SEC26" HREF="gap5_16.html#SEC26">Adjusting the alignment coordinates</A>
<!-- NODE:Editor-Cutoffs --><LI><A NAME="SEC27" HREF="gap5_16.html#SEC27">Adjusting the Cutoff Data</A>
<!-- NODE:Editor-Editing Summary --><LI><A NAME="SEC28" HREF="gap5_17.html">Summary of Editing Commands</A>
</UL>
<!-- NODE:Editor-Select Base --><LI><A NAME="SEC29" HREF="gap5_18.html">Cut and Paste Control of Sequence</A>
<!-- NODE:Editor-Select Seq --><LI><A NAME="SEC30" HREF="gap5_19.html">Selecting Sequences</A>
<!-- NODE:Editor-Annotations --><LI><A NAME="SEC31" HREF="gap5_20.html">Annotations</A>
<UL>
<!-- NODE:Editor-Annotatons-Macro --><LI><A NAME="SEC32" HREF="gap5_20.html#SEC32">Annotation Macros</A>
</UL>
<!-- NODE:Editor-Searching --><LI><A NAME="SEC33" HREF="gap5_21.html">Searching</A>
<UL>
<!-- NODE:Editor-Search-Anno --><LI><A NAME="SEC34" HREF="gap5_21.html#SEC34">Search by Annotation Comments</A>
<!-- NODE:Editor-Search-Type --><LI><A NAME="SEC35" HREF="gap5_21.html#SEC35">Search by Tag Type</A>
<!-- NODE:Editor-Search-UPosition --><LI><A NAME="SEC36" HREF="gap5_21.html#SEC36">Search by Padded Position</A>
<!-- NODE:Editor-Search-PPosition --><LI><A NAME="SEC37" HREF="gap5_21.html#SEC37">Search by Unpadded Position</A>
<!-- NODE:Editor-Search-Seq --><LI><A NAME="SEC38" HREF="gap5_21.html#SEC38">Search by Sequence</A>
<!-- NODE:Editor-Search-Name --><LI><A NAME="SEC39" HREF="gap5_21.html#SEC39">Search by Reading Name</A>
<!-- NODE:Editor-Search-RefIndel --><LI><A NAME="SEC40" HREF="gap5_21.html#SEC40">Search by Reference InDel</A>
<!-- NODE:Editor-Search-ConsQual --><LI><A NAME="SEC41" HREF="gap5_21.html#SEC41">Search by Consensus Quality</A>
<!-- NODE:Editor-Search-ConsDiscrep --><LI><A NAME="SEC42" HREF="gap5_21.html#SEC42">Search by Consensus Discrepancy</A>
<!-- NODE:Editor-Search-ConsHet --><LI><A NAME="SEC43" HREF="gap5_21.html#SEC43">Search by Consensus Heterozygosity</A>
<!-- NODE:Editor-Search-depth-lt --><LI><A NAME="SEC44" HREF="gap5_21.html#SEC44">Search by Low Coverage</A>
<!-- NODE:Editor-Search-depth-ht --><LI><A NAME="SEC45" HREF="gap5_21.html#SEC45">Search by High Coverage</A>
</UL>
<!-- NODE:Editor-Settings --><LI><A NAME="SEC46" HREF="gap5_22.html">The Settings Menu</A>
<UL>
<!-- NODE:Editor-Group Readings --><LI><A NAME="SEC47" HREF="gap5_22.html#SEC47">Group Readings</A>
<!-- NODE:Editor-Disagree --><LI><A NAME="SEC48" HREF="gap5_22.html#SEC48">Highlight Disagreements</A>
<!-- NODE:Editor-Pack Sequences --><LI><A NAME="SEC49" HREF="gap5_22.html#SEC49">Pack Sequences</A>
<!-- NODE:Editor-Hide Annotations --><LI><A NAME="SEC50" HREF="gap5_22.html#SEC50">Hide Annotations</A>
</UL>
<!-- NODE:Editor-Primer Selection --><LI><A NAME="SEC51" HREF="gap5_23.html">Primer Selection</A>
<!-- NODE:Editor-Traces --><LI><A NAME="SEC52" HREF="gap5_24.html">Traces</A>
<!-- NODE:Editor-Info --><LI><A NAME="SEC53" HREF="gap5_25.html">The Editor Information Line</A>
<UL>
<LI><A NAME="SEC54" HREF="gap5_25.html#SEC54">Reading Information</A>
<LI><A NAME="SEC55" HREF="gap5_25.html#SEC55">Contig Information</A>
<LI><A NAME="SEC56" HREF="gap5_25.html#SEC56">Tag Information</A>
</UL>
<!-- NODE:Editor-Joining --><LI><A NAME="SEC57" HREF="gap5_26.html">The Join Editor</A>
<!-- NODE:Editor-Multiple Editors --><LI><A NAME="SEC58" HREF="gap5_27.html">Using Several Editors at Once</A>
<!-- NODE:Editor-Quitting --><LI><A NAME="SEC59" HREF="gap5_28.html">Quitting the Editor</A>
<!-- NODE:Editor-Summary --><LI><A NAME="SEC60" HREF="gap5_29.html">Summary</A>
<UL>
<!-- NODE:Editor-Summary-Keys --><LI><A NAME="SEC61" HREF="gap5_29.html#SEC61">Keyboard summary for editing window</A>
<!-- NODE:Editor-Summary-Mouse --><LI><A NAME="SEC62" HREF="gap5_29.html#SEC62">Mouse summary for editing window</A>
<!-- NODE:Editor-Summary-MouseNames --><LI><A NAME="SEC63" HREF="gap5_29.html#SEC63">Mouse summary for names window</A>
</UL>
<!-- NODE:Restrict --><LI><A NAME="SEC64" HREF="gap5_30.html">Plotting Restriction Enzymes</A>
<UL>
<!-- NODE:Restrict-Selecting --><LI><A NAME="SEC65" HREF="gap5_31.html">Selecting Enzymes</A>
<!-- NODE:Restrict-Examining --><LI><A NAME="SEC66" HREF="gap5_32.html">Examining the Plot</A>
<!-- NODE:Restrict-Reconfig --><LI><A NAME="SEC67" HREF="gap5_33.html">Reconfiguring the Plot</A>
<!-- NODE:Restrict-Output --><LI><A NAME="SEC68" HREF="gap5_34.html">Textual Outputs</A>
</UL>
</UL>
<LI><A NAME="SEC69" HREF="gap5_34.html#SEC69">Importing and Exporting Data</A>
<UL>
<LI><A NAME="SEC70" HREF="gap5_34.html#SEC70">Assembly</A>
<!-- NODE:Assembly-tg_index --><LI><A NAME="SEC71" HREF="gap5_36.html">Importing with tg_index</A>
<!-- NODE:Assembly-Fasta --><LI><A NAME="SEC72" HREF="gap5_37.html">Importing fasta/fastq files</A>
<!-- NODE:Assembly-Map-bwa-aln --><LI><A NAME="SEC73" HREF="gap5_38.html">Mapped assembly by bwa aln</A>
<!-- NODE:Assembly-Map-bwa-dbwtsw --><LI><A NAME="SEC74" HREF="gap5_38.html#SEC74">Mapped assembly by bwa dbwtsw</A>
<!-- NODE:ImportGFF --><LI><A NAME="SEC75" HREF="gap5_39.html">Importing GFF</A>
<!-- NODE:ExportTags --><LI><A NAME="SEC76" HREF="gap5_40.html">Export Tags</A>
<!-- NODE:ExportSequences --><LI><A NAME="SEC77" HREF="gap5_41.html">Export Sequences</A>
</UL>
<!-- NODE:Matches --><LI><A NAME="SEC78" HREF="gap5_42.html">Finding Sequence Matches</A>
<UL>
<!-- NODE:FIJ --><LI><A NAME="SEC79" HREF="gap5_42.html#SEC79">Find Internal Joins</A>
<UL>
<!-- NODE:FIJ-Dialogue --><LI><A NAME="SEC80" HREF="gap5_43.html">Find Internal Joins Dialogue</A>
</UL>
<!-- NODE:Repeats --><LI><A NAME="SEC81" HREF="gap5_44.html">Find Repeats</A>
<!-- NODE:Read Pairs --><LI><A NAME="SEC82" HREF="gap5_45.html">Find Read Pairs</A>
<UL>
<!-- NODE:ReadPair-Display --><LI><A NAME="SEC83" HREF="gap5_46.html">Find Read Pairs Graphical Output</A>
</UL>
<!-- NODE:Find Oligos --><LI><A NAME="SEC84" HREF="gap5_47.html">Sequence Search</A>
</UL>
<!-- NODE:Disassembly and Data Deletion --><LI><A NAME="SEC85" HREF="gap5_48.html">Checking Assemblies and Removing Readings</A>
<UL>
<UL>
<!-- NODE:Check Assembly --><LI><A NAME="SEC86" HREF="gap5_49.html">Checking Assemblies</A>
</UL>
<!-- NODE:Removing Readings --><LI><A NAME="SEC87" HREF="gap5_50.html">Removing Readings and Breaking Contigs</A>
<UL>
<!-- NODE:Break Contig --><LI><A NAME="SEC88" HREF="gap5_51.html">Breaking Contigs</A>
<!-- NODE:Disassemble --><LI><A NAME="SEC89" HREF="gap5_52.html">Disassembling Readings</A>
<!-- NODE:Delete Contigs --><LI><A NAME="SEC90" HREF="gap5_53.html">Delete Contigs</A>
</UL>
</UL>
<!-- NODE:Tidying up alignments --><LI><A NAME="SEC91" HREF="gap5_54.html">Tidying up alignments</A>
<UL>
<!-- NODE:Shuffle Pads --><LI><A NAME="SEC92" HREF="gap5_54.html#SEC92">Shuffle Pads</A>
<!-- NODE:Remove Pad Columns --><LI><A NAME="SEC93" HREF="gap5_55.html">Remove Pad Columns</A>
<!-- NODE:Remove Contig Holes --><LI><A NAME="SEC94" HREF="gap5_56.html">Remove Contig Holes</A>
</UL>
<!-- NODE:Calculate Consensus --><LI><A NAME="SEC95" HREF="gap5_57.html">Calculating Consensus Sequences</A>
<UL>
<!-- NODE:Con-Normal --><LI><A NAME="SEC96" HREF="gap5_58.html">Normal Consensus Output</A>
<!-- NODE:Con-Calculation --><LI><A NAME="SEC97" HREF="gap5_59.html">The Consensus Algorithms</A>
<UL>
<!-- NODE:Con-Calculation-1 --><LI><A NAME="SEC98" HREF="gap5_60.html">Consensus Calculation Using Base Frequencies</A>
<!-- NODE:Con-Calculation-2 --><LI><A NAME="SEC99" HREF="gap5_61.html">Consensus Calculation Using Weighted Base Frequencies</A>
<!-- NODE:Con-Calculation-3 --><LI><A NAME="SEC100" HREF="gap5_62.html">Consensus Calculation Using Confidence values</A>
<!-- NODE:Qual-Cal --><LI><A NAME="SEC101" HREF="gap5_63.html">The Quality Calculation</A>
</UL>
<!-- NODE:Con-Evaluation --><LI><A NAME="SEC102" HREF="gap5_64.html">List Consensus Confidence</A>
<!-- NODE:Con-ListBaseConf --><LI><A NAME="SEC103" HREF="gap5_65.html">List Base Confidence</A>
</UL>
<!-- NODE:Misc --><LI><A NAME="SEC104" HREF="gap5_66.html">Other Miscellany</A>
<!-- NODE:List Libraries --><LI><A NAME="SEC105" HREF="gap5_67.html">List Libraries</A>
<!-- NODE:Results --><LI><A NAME="SEC106" HREF="gap5_68.html">Results Manager</A>
<!-- NODE:Lists --><LI><A NAME="SEC107" HREF="gap5_69.html">Lists</A>
<UL>
<!-- NODE:List-Special --><LI><A NAME="SEC108" HREF="gap5_70.html">Special List Names</A>
<!-- NODE:List-Commands --><LI><A NAME="SEC109" HREF="gap5_71.html">Basic List Commands</A>
<!-- NODE:List-ContigToRead --><LI><A NAME="SEC110" HREF="gap5_72.html">Contigs To Readings Command</A>
<!-- NODE:List-SearchSequenceNames --><LI><A NAME="SEC111" HREF="gap5_73.html">Search Sequence Names</A>
</UL>
<!-- NODE:Index --><LI><A NAME="SEC112" HREF="gap5_74.html">Index</A>
</UL>
<hr size=4>
<a href="../index.html"><img src="i/nav_home.gif" alt="home"></a>
<a href="master_brief.html"><img src="i/nav_brief.gif" alt="brief"></a>
<a href="master_contents.html"><img src="i/nav_full.gif" alt="full"></a>
<hr>
<P>This document was generated on 25 April 2016 using the
<A HREF="http://wwwcn.cern.ch/dci/texi2html/">texi2html</A>
translator version 1.39 (with local modifications)</P>
<hr>
<i>Last generated on 25 April 2016.</i>
<font size="-1"><br>
</font>
</BODY>
</HTML>
