
<HTML>
<HEAD>
<!-- This HTML file has been created by texi2html 1.39
     from gap5.htmlinfo on 25 April 2016 -->

<TITLE>Gap5 - Assembly-Fasta</TITLE>
</HEAD>
<BODY bgcolor="#ffffff">
<A HREF="gap5_1.html"><IMG SRC="i/nav_first.gif" ALT="first"></A>  <A HREF="gap5_36.html"><IMG SRC="i/nav_prev.gif" ALT="previous"></A>  <A HREF="gap5_38.html"><IMG SRC="i/nav_next.gif" ALT="next"></A>  <A HREF="gap5_74.html"><IMG SRC="i/nav_last.gif" ALT="last"></A>  <A HREF="gap5_toc.html"><IMG SRC="i/nav_top.gif" ALT="contents"></A>
<HR>
<!-- NODE:Assembly-Fasta -->


<H2><A NAME="SEC72" HREF="gap5_toc.html#SEC72">Importing fasta/fastq files</A></H2>
<P>
<A NAME="IDX114"></A>

</P>
<P>
Sometimes we have a few individual sequences we wish to import as
single-read contigs. That is we won't align them against each other or
against existing data, but just load them into our gap5 database so we
can then run tools such as Find Repeats or Find Internal Joins on
them. (This can be ideal for importing consensus sequences.)

</P>
<P>
The "Import Fasta/Fastq as single-read contigs" function is designed
for this purpose.  Behind the scenes it is nothing more than running
<CODE>tg_index -a</CODE> to add a fasta or fastq file.

</P>
<HR>
<A HREF="gap5_1.html"><IMG SRC="i/nav_first.gif" ALT="first"></A>  <A HREF="gap5_36.html"><IMG SRC="i/nav_prev.gif" ALT="previous"></A>  <A HREF="gap5_38.html"><IMG SRC="i/nav_next.gif" ALT="next"></A>  <A HREF="gap5_74.html"><IMG SRC="i/nav_last.gif" ALT="last"></A>  <A HREF="gap5_toc.html"><IMG SRC="i/nav_top.gif" ALT="contents"></A>
<hr>
<i>Last generated on 25 April 2016.</i>
<font size="-1"><br>
</font>
</BODY>
</HTML>
