<HTML>
<HEAD>
<TITLE>Master Index</TITLE>
</HEAD>
<BODY bgcolor="#ffffff">
<a href="../index.html"><img src="i/nav_home.gif" alt="home"></a>
<a href="master_brief.html"><img src="i/nav_brief.gif" alt="brief"></a>
<a href="master_contents.html"><img src="i/nav_full.gif" alt="full"></a>
<hr size=4>
<H1>Master Index</H1>
<H3>For version 1.5 (2004)</H3>
<P>
<A HREF="master_index.html#LET-">-</A>
<A HREF="master_index.html#LET.">.</A>
<A HREF="master_index.html#LET2">2</A>
<A HREF="master_index.html#LET3">3</A>
<A HREF="master_index.html#LET6">6</A>
<A HREF="master_index.html#LET_">_</A>
<A HREF="master_index.html#LETa">A</A>
<A HREF="master_index.html#LETb">B</A>
<A HREF="master_index.html#LETc">C</A>
<A HREF="master_index.html#LETd">D</A>
<A HREF="master_index.html#LETe">E</A>
<A HREF="master_index.html#LETf">F</A>
<A HREF="master_index.html#LETg">G</A>
<A HREF="master_index.html#LETh">H</A>
<A HREF="master_index.html#LETi">I</A>
<A HREF="master_index.html#LETj">J</A>
<A HREF="master_index.html#LETk">K</A>
<A HREF="master_index.html#LETl">L</A>
<A HREF="master_index.html#LETm">M</A>
<A HREF="master_index.html#LETn">N</A>
<A HREF="master_index.html#LETo">O</A>
<A HREF="master_index.html#LETp">P</A>
<A HREF="master_index.html#LETq">Q</A>
<A HREF="master_index.html#LETr">R</A>
<A HREF="master_index.html#LETs">S</A>
<A HREF="master_index.html#LETt">T</A>
<A HREF="master_index.html#LETu">U</A>
<A HREF="master_index.html#LETv">V</A>
<A HREF="master_index.html#LETw">W</A>
<A HREF="master_index.html#LETy">Y</A>
<A HREF="master_index.html#LETz">Z</A>
<P>
<A NAME="LET-"></A><H2>-</H2>
<DIR>
<LI><A HREF="gap4_181.html#IDX639">-bitsize</A><LI><A HREF="gap4_181.html#IDX648">-check</A><LI><A HREF="gap4_181.html#IDX657">-csel</A><LI><A HREF="gap4_181.html#IDX652">-exec_notes</A><LI><A HREF="gap4_181.html#IDX642">-maxdb</A><LI><A HREF="gap4_181.html#IDX644">-maxseq</A><LI><A HREF="gap4_181.html#IDX658">-no_csel</A><LI><A HREF="gap4_181.html#IDX653">-no_exec_notes</A><LI><A HREF="gap4_181.html#IDX656">-no_rawdata_note</A><LI><A HREF="gap4_181.html#IDX649">-nocheck</A><LI><A HREF="gap4_181.html#IDX655">-rawdata_note</A><LI><A HREF="gap4_181.html#IDX646">-read_only</A></DIR>
<A NAME="LET."></A><H2>.</H2>
<DIR>
<LI><A HREF="pregap4_86.html#IDX299">.assembly/assemble.dat</A><LI><A HREF="pregap4_86.html#IDX298">.assembly/assemble_stderr</A><LI><A HREF="pregap4_84.html#IDX272">.assembly/cap2_stderr</A><LI><A HREF="pregap4_84.html#IDX271">.assembly/cap2_stdout</A><LI><A HREF="pregap4_85.html#IDX277">.assembly/cap3_stderr</A><LI><A HREF="pregap4_85.html#IDX276">.assembly/cap3_stdout</A><LI><A HREF="pregap4_86.html#IDX296">.assembly/constraints.ascii</A><LI><A HREF="pregap4_86.html#IDX297">.assembly/constraints.dat</A><LI><A HREF="pregap4_86.html#IDX295">.assembly/constraints_stderr</A><LI><A HREF="pregap4_88.html#IDX321">.assembly/fofn</A><LI><A HREF="pregap4_86.html#IDX294">.assembly/graph.dat</A><LI><A HREF="pregap4_86.html#IDX293">.assembly/graph_stderr</A><LI><A HREF="pregap4_87.html#IDX309">.assembly/phrap_stderr</A><LI><A HREF="pregap4_87.html#IDX308">.assembly/phrap_stdout</A><LI><A HREF="pregap4_86.html#IDX301">.assembly/write_exp_file_stderr</A><LI><A HREF="pregap4_86.html#IDX300">.assembly/write_exp_file_stdout</A><LI><A HREF="pregap4_88.html#IDX322">.aux</A><LI><A HREF="pregap4_77.html#IDX221">.blast</A><LI><A HREF="pregap4_85.html#IDX280">.cap3_info</A><LI><A HREF="pregap4_85.html#IDX279">.con.results</A><LI><A HREF="pregap4_85.html#IDX278">.con</A><LI><A HREF="pregap4_87.html#IDX311">.contigs.qual</A><LI><A HREF="pregap4_87.html#IDX310">.contigs</A><LI><A HREF="pregap4_74.html#IDX202">.cvec_failed</A><LI><A HREF="pregap4_74.html#IDX201">.cvec_passed</A><LI><A HREF="pregap4_90.html#IDX331">.failed</A><LI><A HREF="pregap4_81.html#IDX243">.fasta.cat</A><LI><A HREF="pregap4_81.html#IDX245">.fasta.masked.log</A><LI><A HREF="pregap4_81.html#IDX244">.fasta.masked</A><LI><A HREF="pregap4_81.html#IDX247">.fasta.out.xm</A><LI><A HREF="pregap4_81.html#IDX246">.fasta.out</A><LI><A HREF="pregap4_81.html#IDX248">.fasta.tbl</A><LI><A HREF="gap4_170.html#IDX605">.gaprc</A><LI><A HREF="pregap4_90.html#IDX332">.log</A><LI><A HREF="pregap4_90.html#IDX330">.passed</A><LI><A HREF="pregap4_87.html#IDX313">.phrap_log</A><LI><A HREF="pregap4_56.html#IDX109">.pregap4rc</A><LI><A HREF="pregap4_90.html#IDX333">.report</A><LI><A HREF="pregap4_64.html#IDX131">.scf_dir</A><LI><A HREF="pregap4_76.html#IDX217">.screenseq_failed</A><LI><A HREF="pregap4_76.html#IDX216">.screenseq_passed</A><LI><A HREF="pregap4_75.html#IDX209">.screenvec_failed</A><LI><A HREF="pregap4_75.html#IDX208">.screenvec_passed</A><LI><A HREF="pregap4_87.html#IDX312">.singlets</A><LI><A HREF="pregap4_71.html#IDX178">.svec_failed</A><LI><A HREF="pregap4_71.html#IDX177">.svec_passed</A><LI><A HREF="pregap4_80.html#IDX231">.tagrep_free</A><LI><A HREF="pregap4_80.html#IDX233">.tagrep_log</A><LI><A HREF="pregap4_80.html#IDX232">.tagrep_repeat</A><LI><A HREF="gap4_170.html#IDX607">.tk_utilsrc</A></DIR>
<A NAME="LET2"></A><H2>2</H2>
<DIR>
<LI><A HREF="gap4_40.html#IDX93">2nd-Highest Confidence</A></DIR>
<A NAME="LET3"></A><H2>3</H2>
<DIR>
<LI><A HREF="gap4_63.html#IDX269">3 Character Amino Acids: contig editor</A></DIR>
<A NAME="LET6"></A><H2>6</H2>
<DIR>
<LI><A HREF="gap4_181.html#IDX641">64-bit Gap4 databases</A></DIR>
<A NAME="LET_"></A><H2>_</H2>
<DIR>
<LI><A HREF="pregap4_68.html#IDX141">_com</A></DIR>
<A NAME="LETa"></A><H2>a</H2>
<DIR>
<LI><A HREF="formats_20.html#IDX43">AC: experiment file line type</A><LI><A HREF="spin_49.html#IDX165">Active sequence: spin</A><LI><A HREF="gap4_20.html#IDX32">Active tags</A><LI><A HREF="pregap4_56.html#IDX106">Adding modules</A><LI><A HREF="pregap4_62.html#IDX124">ALF/ABI to SCF conversion configuration</A><LI><A HREF="pregap4_45.html#IDX85">ALF/ABI to SCF conversion module</A><LI><A HREF="spin_38.html#IDX92">Align sequences: spin</A><LI><A HREF="gap4_62.html#IDX225">Align: contig editor</A><LI><A HREF="gap4_62.html#IDX209">aligned readings: printing</A><LI><A HREF="gap4_62.html#IDX208">aligned readings: saving to file</A><LI><A HREF="gap4_103.html#IDX405">aligned readings: sorted on alignment score</A><LI><A HREF="gap5_49.html#IDX154">aligned readings: sorted on alignment score</A><LI><A HREF="spin_39.html#IDX94">Alignment local: spin</A><LI><A HREF="gap4_176.html#IDX620">Alignment matrix file</A><LI><A HREF="gap4_176.html#IDX619">Alignment scores</A><LI><A HREF="gap4_140.html#IDX481">allcontigs list</A><LI><A HREF="gap5_70.html#IDX193">allcontigs list</A><LI><A HREF="gap4_56.html#IDX137">Allow del any in cons: contig editor</A><LI><A HREF="gap4_56.html#IDX136">Allow del dash cons: contig editor</A><LI><A HREF="gap4_56.html#IDX132">Allow del in read: contig editor</A><LI><A HREF="gap4_56.html#IDX153">Allow F12 for fast tag deletion: contig editor</A><LI><A HREF="gap4_56.html#IDX135">Allow insert any in cons: contig editor</A><LI><A HREF="gap4_56.html#IDX131">Allow insert in read: contig editor</A><LI><A HREF="gap4_56.html#IDX143">Allow reading shift: contig editor</A><LI><A HREF="gap4_56.html#IDX141">Allow replace in cons: contig editor</A><LI><A HREF="gap4_56.html#IDX145">Allow transpose any: contig editor</A><LI><A HREF="gap4_56.html#IDX147">Allow uppercase: contig editor</A><LI><A HREF="gap4_140.html#IDX482">allreadings list</A><LI><A HREF="gap5_70.html#IDX194">allreadings list</A><LI><A HREF="interface_8.html#IDX16">Alt left mouse button: overview</A><LI><A HREF="pregap4_81.html#IDX237">alu_only</A><LI><A HREF="gap4_18.html#IDX28">Annotating contigs</A><LI><A HREF="gap4_18.html#IDX27">Annotating readings</A><LI><A HREF="gap4_178.html#IDX632">Annotation Selector</A><LI><A HREF="gap4_163.html#IDX580">Annotation structure: doctor database</A><LI><A HREF="gap4_147.html#IDX514">Annotations, searching for</A><LI><A HREF="gap4_60.html#IDX171">Annotations: contig editor</A><LI><A HREF="gap5_20.html#IDX58">Annotations: contig editor</A><LI><A HREF="gap4_166.html#IDX595">Annotations: deleting (Doctor Database)</A><LI><A HREF="gap4_127.html#IDX446">annotations: entering from a file</A><LI><A HREF="gap4_166.html#IDX596">Annotations: outputting to file (Doctor Database)</A><LI><A HREF="formats_20.html#IDX44">AP: experiment file line type</A><LI><A HREF="formats_20.html#IDX45">AQ: experiment file line type</A><LI><A HREF="pregap4_86.html#IDX290">assem_d_threshold</A><LI><A HREF="pregap4_86.html#IDX288">assem_e_rate</A><LI><A HREF="pregap4_86.html#IDX287">assem_number</A><LI><A HREF="pregap4_86.html#IDX289">assem_o_threshold</A><LI><A HREF="gap4_86.html#IDX354">Assemble: independently i.e. ignoring previous data</A><LI><A HREF="gap4_102.html#IDX399">assembly problems: breaking contigs</A><LI><A HREF="gap5_48.html#IDX148">assembly problems: breaking contigs</A><LI><A HREF="gap4_102.html#IDX401">assembly problems: disassembling readings</A><LI><A HREF="gap5_48.html#IDX150">assembly problems: disassembling readings</A><LI><A HREF="gap4_102.html#IDX400">assembly problems: removing readings</A><LI><A HREF="gap5_48.html#IDX149">assembly problems: removing readings</A><LI><A HREF="gap5_38.html#IDX115">Assembly: bwa aln</A><LI><A HREF="gap5_38.html#IDX117">Assembly: bwa dbwtsw</A><LI><A HREF="gap4_90.html#IDX361">Assembly: directed</A><LI><A HREF="gap4_93.html#IDX367">Assembly: failure codes</A><LI><A HREF="gap5_37.html#IDX114">Assembly: fasta/fastq</A><LI><A HREF="gap4_89.html#IDX359">Assembly: into new contigs</A><LI><A HREF="gap4_88.html#IDX357">Assembly: into one contig</A><LI><A HREF="gap4_89.html#IDX360">Assembly: into separate contigs</A><LI><A HREF="gap4_84.html#IDX346">Assembly: large projects</A><LI><A HREF="gap4_84.html#IDX342">Assembly: limits</A><LI><A HREF="gap4_84.html#IDX345">Assembly: maxdb</A><LI><A HREF="gap4_84.html#IDX344">Assembly: maxseq</A><LI><A HREF="gap4_84.html#IDX343">Assembly: resetting limits</A><LI><A HREF="gap4_91.html#IDX363">Assembly: screen only</A><LI><A HREF="gap4_85.html#IDX352">Assembly: shotgun</A><LI><A HREF="gap4_87.html#IDX355">Assembly: single stranded regions</A><LI><A HREF="gap4_88.html#IDX358">Assembly: stack readings</A><LI><A HREF="gap5_36.html#IDX112">Assembly: tg_index</A><LI><A HREF="gap4_92.html#IDX366">Assembly: tips</A><LI><A HREF="gap4_84.html#IDX340">Assembly</A><LI><A HREF="gap5_35.html#IDX110">Assembly</A><LI><A HREF="pregap4_65.html#IDX133">ATQA configuration</A><LI><A HREF="pregap4_18.html#IDX28">ATQA module</A><LI><A HREF="pregap4_65.html#IDX134">atqa.p4m</A><LI><A HREF="pregap4_68.html#IDX139">Augment Experiment files configuration</A><LI><A HREF="pregap4_21.html#IDX35">Augment Experiment files module</A><LI><A HREF="pregap4_55.html#IDX105">Augment, by line types</A><LI><A HREF="pregap4_54.html#IDX102">Augment, by text database</A><LI><A HREF="pregap4_68.html#IDX140">augment_exp.p4m</A><LI><A HREF="spin_30.html#IDX69">Author test:spin</A><LI><A HREF="gap4_63.html#IDX251">Auto-diff traces: contig editor</A><LI><A HREF="gap4_63.html#IDX246">Auto-display Traces: contig editor</A><LI><A HREF="gap4_63.html#IDX267">Auto-save: contig editor</A><LI><A HREF="formats_20.html#IDX46">AV: experiment file line type</A></DIR>
<A NAME="LETb"></A><H2>b</H2>
<DIR>
<LI><A HREF="gap4_156.html#IDX543">Backing up databases</A><LI><A HREF="pregap4_82.html#IDX254">band_width</A><LI><A HREF="gap4_16.html#IDX21">Base accuracies - use of</A><LI><A HREF="spin_13.html#IDX5">Base composition plotting:spin</A><LI><A HREF="formats_6.html#IDX23">Base: SCF structure</A><LI><A HREF="gap4_71.html#IDX321">BASE_BRIEF_FORMAT1</A><LI><A HREF="gap5_25.html#IDX88">BASE_BRIEF_FORMAT1</A><LI><A HREF="gap4_71.html#IDX322">BASE_BRIEF_FORMAT2</A><LI><A HREF="gap5_25.html#IDX91">BASE_BRIEF_FORMAT2</A><LI><A HREF="formats_20.html#IDX47">BC: experiment file line type</A><LI><A HREF="pregap4_62.html#IDX126">bit_size</A><LI><A HREF="gap4_181.html#IDX640">bitsize (command line option)</A><LI><A HREF="pregap4_77.html#IDX218">Blast screen configuration</A><LI><A HREF="pregap4_28.html#IDX49">Blast screen module</A><LI><A HREF="pregap4_77.html#IDX219">blast.p4m</A><LI><A HREF="gap4_62.html#IDX227">Break contig: contig editor</A><LI><A HREF="gap4_105.html#IDX406">Break contig</A><LI><A HREF="gap5_51.html#IDX155">Break contig</A><LI><A HREF="gap4_152.html#IDX530">Busy file</A><LI><A HREF="gap4_4.html#IDX2">BUSY files</A><LI><A HREF="interface_8.html#IDX13">Buttons: mouse overview</A><LI><A HREF="interface_4.html#IDX3">Buttons</A><LI><A HREF="gap5_38.html#IDX118">bwa</A><LI><A HREF="formats_10.html#IDX31">Byte ordering: SCF</A></DIR>
<A NAME="LETc"></A><H2>c</H2>
<DIR>
<LI><A HREF="gap4_118.html#IDX432">Calculate consensus: algorithm</A><LI><A HREF="gap5_59.html#IDX170">Calculate consensus: algorithm</A><LI><A HREF="gap4_123.html#IDX438">Calculate consensus: confidence</A><LI><A HREF="gap5_64.html#IDX176">Calculate consensus: confidence</A><LI><A HREF="gap4_115.html#IDX425">Calculate consensus: extended consensus</A><LI><A HREF="gap4_114.html#IDX422">Calculate consensus: normal consensus</A><LI><A HREF="gap5_58.html#IDX167">Calculate consensus: normal consensus</A><LI><A HREF="gap4_117.html#IDX429">Calculate consensus: quality</A><LI><A HREF="gap4_123.html#IDX437">Calculate consensus: reliability</A><LI><A HREF="gap5_64.html#IDX175">Calculate consensus: reliability</A><LI><A HREF="gap4_116.html#IDX427">Calculate consensus: unfinished consensus</A><LI><A HREF="gap4_113.html#IDX419">Calculate consensus</A><LI><A HREF="gap5_57.html#IDX164">Calculate consensus</A><LI><A HREF="pregap4_84.html#IDX268">Cap2 assembly configuration</A><LI><A HREF="pregap4_38.html#IDX71">Cap2 assembly module</A><LI><A HREF="pregap4_84.html#IDX269">cap2_assemble.p4m</A><LI><A HREF="pregap4_85.html#IDX273">Cap3 assembly configuration</A><LI><A HREF="pregap4_39.html#IDX73">Cap3 assembly module</A><LI><A HREF="pregap4_85.html#IDX274">cap3_assemble.p4m</A><LI><A HREF="formats_20.html#IDX48">CC: experiment file line type</A><LI><A HREF="formats_20.html#IDX49">CF: experiment file line type</A><LI><A HREF="pregap4_74.html#IDX200">CF</A><LI><A HREF="formats_20.html#IDX50">CH: experiment file line type</A><LI><A HREF="gap5_2.html#IDX4">Change directory</A><LI><A HREF="spin_40.html#IDX102">Changing the default number of matches: spin</A><LI><A HREF="spin_40.html#IDX101">Changing the maximum number of matches: spin</A><LI><A HREF="spin_40.html#IDX105">Changing the score matrix: spin</A><LI><A HREF="gap4_181.html#IDX651">check (command line option)</A><LI><A HREF="gap4_103.html#IDX402">Check assembly</A><LI><A HREF="gap5_49.html#IDX151">Check assembly</A><LI><A HREF="gap4_161.html#IDX568">Check database: annotation checks</A><LI><A HREF="gap4_161.html#IDX572">Check database: clone checks</A><LI><A HREF="gap4_161.html#IDX566">Check database: contig checks</A><LI><A HREF="gap4_161.html#IDX565">Check database: database checks</A><LI><A HREF="gap4_164.html#IDX589">Check database: ignoring</A><LI><A HREF="gap4_161.html#IDX569">Check database: note checks</A><LI><A HREF="gap4_161.html#IDX567">Check database: reading checks</A><LI><A HREF="gap4_161.html#IDX570">Check database: template checks</A><LI><A HREF="gap4_161.html#IDX571">Check database: vector checks</A><LI><A HREF="gap4_160.html#IDX564">Check database</A><LI><A HREF="gap5_2.html#IDX5">Check database</A><LI><A HREF="pregap4_91.html#IDX343">check_params</A><LI><A HREF="formats_14.html#IDX34">Chunks, ZTR</A><LI><A HREF="spin_49.html#IDX171">circular sequences:spin</A><LI><A HREF="formats_20.html#IDX51">CL: experiment file line type</A><LI><A HREF="interface_9.html#IDX23">Clear: in output window</A><LI><A HREF="pregap4_70.html#IDX155">clip_mode</A><LI><A HREF="gap4_134.html#IDX465">Clipping by differences</A><LI><A HREF="gap4_137.html#IDX471">Clipping by N bases</A><LI><A HREF="gap4_136.html#IDX469">Clipping by quality, ends only</A><LI><A HREF="gap4_135.html#IDX467">Clipping by quality</A><LI><A HREF="read_clipping_2.html#IDX1">clipping readings</A><LI><A HREF="gap4_133.html#IDX464">Clipping within Gap4</A><LI><A HREF="gap4_163.html#IDX584">Clone structure: doctor database</A><LI><A HREF="vector_clip_9.html#IDX9">Cloning site, defining</A><LI><A HREF="vector_clip_10.html#IDX16">Cloning site, finding</A><LI><A HREF="pregap4_74.html#IDX194">Cloning vector clip configuration (old style)</A><LI><A HREF="pregap4_73.html#IDX185">Cloning vector clip configuration</A><LI><A HREF="pregap4_44.html#IDX83">Cloning vector clip module (old style)</A><LI><A HREF="pregap4_25.html#IDX43">Cloning vector clip module</A><LI><A HREF="pregap4_73.html#IDX186">cloning_vector_clip.p4m</A><LI><A HREF="gap4_151.html#IDX521">CLOS note type</A><LI><A HREF="spin_33.html#IDX84">Cloverleaf:spin</A><LI><A HREF="formats_20.html#IDX52">CN: experiment file line type</A><LI><A HREF="spin_14.html#IDX15">Codon composition:spin</A><LI><A HREF="spin_14.html#IDX13">Codon frequencies:spin</A><LI><A HREF="spin_14.html#IDX14">Codon tables:spin</A><LI><A HREF="spin_28.html#IDX62">Codon usage method:spin</A><LI><A HREF="spin_30.html#IDX65">Codon usage tables:spin</A><LI><A HREF="spin_14.html#IDX12">Codon usage:spin</A><LI><A HREF="gap4_170.html#IDX609">Colour blindness</A><LI><A HREF="interface_11.html#IDX27">Colour selector</A><LI><A HREF="gap4_63.html#IDX262">Colour: contig editor highlight disagreements</A><LI><A HREF="gap5_22.html#IDX75">Colour: contig editor highlight disagreements</A><LI><A HREF="pregap4_13.html#IDX17">Colours</A><LI><A HREF="trev_2.html#IDX5">Command line arguments: Trev</A><LI><A HREF="gap4_181.html#IDX638">Command line arguments</A><LI><A HREF="gap4_62.html#IDX197">Commands menu: contig editor</A><LI><A HREF="formats_7.html#IDX25">Comments: SCF</A><LI><A HREF="gap4_25.html#IDX51">Comparator window</A><LI><A HREF="gap5_7.html#IDX20">Comparator window</A><LI><A HREF="gap4_63.html#IDX264">Compare Strands: contig editor</A><LI><A HREF="gap4_95.html#IDX373">Complement contig</A><LI><A HREF="spin_49.html#IDX172">Complement sequence: spin</A><LI><A HREF="pregap4_59.html#IDX117">Component configuration</A><LI><A HREF="pregap4_52.html#IDX96">Components</A><LI><A HREF="spin_13.html#IDX7">Composition: sequence:spin</A><LI><A HREF="gap4_111.html#IDX414">Compressions: suggested experiments</A><LI><A HREF="pregap4_70.html#IDX154">conf_val</A><LI><A HREF="gap4_63.html#IDX288">Confidence in contig editor</A><LI><A HREF="gap4_124.html#IDX442">Confidence of base calls</A><LI><A HREF="gap5_65.html#IDX180">Confidence of base calls</A><LI><A HREF="gap4_123.html#IDX440">Confidence of consensus</A><LI><A HREF="gap5_64.html#IDX178">Confidence of consensus</A><LI><A HREF="gap4_16.html#IDX22">Confidence values - use of</A><LI><A HREF="gap4_36.html#IDX89">Confidence values graph</A><LI><A HREF="gap4_77.html#IDX333">Confidence values: editing techniques</A><LI><A HREF="pregap4_4.html#IDX6">configuration files: pregap4</A><LI><A HREF="pregap4_46.html#IDX86">Configuration files</A><LI><A HREF="pregap4_57.html#IDX113">Configuration: pregap4 low level</A><LI><A HREF="gap4_174.html#IDX614">Configure menus</A><LI><A HREF="gap4_63.html#IDX232">configure: contig editor</A><LI><A HREF="gap5_22.html#IDX66">configure: contig editor</A><LI><A HREF="spin_21.html#IDX26">Configure: restriction enzymes: spin</A><LI><A HREF="gap4_48.html#IDX109">Configure: restriction enzymes</A><LI><A HREF="gap5_33.html#IDX105">Configure: restriction enzymes</A><LI><A HREF="pregap4_91.html#IDX345">configure_dialogue</A><LI><A HREF="pregap4_60.html#IDX119">Configuring modules</A><LI><A HREF="pregap4_13.html#IDX15">Configuring pregap4</A><LI><A HREF="interface_15.html#IDX40">Configuring: fonts</A><LI><A HREF="gap4_63.html#IDX255">Consensus algorith in contig editor</A><LI><A HREF="gap4_123.html#IDX439">Consensus calculation confidence</A><LI><A HREF="gap5_64.html#IDX177">Consensus calculation confidence</A><LI><A HREF="gap4_118.html#IDX433">Consensus calculation method</A><LI><A HREF="gap5_59.html#IDX171">Consensus calculation method</A><LI><A HREF="gap4_61.html#IDX196">Consensus discrepancies: searching for in contig editor</A><LI><A HREF="gap4_118.html#IDX434">consensus IUB codes</A><LI><A HREF="gap5_59.html#IDX172">consensus IUB codes</A><LI><A HREF="gap4_62.html#IDX213">Consensus Trace</A><LI><A HREF="gap4_63.html#IDX231">Consensus: contig editor</A><LI><A HREF="gap5_22.html#IDX65">Consensus: contig editor</A><LI><A HREF="gap4_113.html#IDX418">Consensus: outputting</A><LI><A HREF="gap5_57.html#IDX163">Consensus: outputting</A><LI><A HREF="spin_33.html#IDX85">Conserved bases in tRNA:spin</A><LI><A HREF="gap4_35.html#IDX88">Consistency display</A><LI><A HREF="gap4_62.html#IDX228">Contig breaking</A><LI><A HREF="gap4_27.html#IDX60">Contig comparator: auto navigation</A><LI><A HREF="gap5_9.html#IDX28">Contig comparator: auto navigation</A><LI><A HREF="gap4_26.html#IDX53">Contig Comparator: manipulating results</A><LI><A HREF="gap5_8.html#IDX22">Contig Comparator: manipulating results</A><LI><A HREF="gap4_27.html#IDX61">Contig comparator: next button</A><LI><A HREF="gap5_9.html#IDX29">Contig comparator: next button</A><LI><A HREF="gap4_25.html#IDX52">Contig Comparator</A><LI><A HREF="gap5_7.html#IDX21">Contig Comparator</A><LI><A HREF="gap4_95.html#IDX374">Contig complementing</A><LI><A HREF="gap4_63.html#IDX270">Contig Editor: 3 Character Amino Acids</A><LI><A HREF="gap4_62.html#IDX224">Contig Editor: align</A><LI><A HREF="gap5_16.html#IDX47">Contig Editor: alignment coordinates</A><LI><A HREF="gap4_56.html#IDX140">Contig Editor: allow del any in cons</A><LI><A HREF="gap4_56.html#IDX139">Contig Editor: allow del dash in cons</A><LI><A HREF="gap4_56.html#IDX134">Contig Editor: Allow del in read</A><LI><A HREF="gap4_56.html#IDX154">Contig Editor: Allow F12 for fast tag deletion</A><LI><A HREF="gap4_56.html#IDX138">Contig Editor: allow insert any in cons</A><LI><A HREF="gap4_56.html#IDX133">Contig Editor: allow insert in read</A><LI><A HREF="gap4_56.html#IDX144">Contig Editor: allow reading shift</A><LI><A HREF="gap4_56.html#IDX142">Contig Editor: allow replace in cons</A><LI><A HREF="gap4_56.html#IDX146">Contig Editor: allow transpose any</A><LI><A HREF="gap4_56.html#IDX148">Contig Editor: allow uppercase</A><LI><A HREF="gap4_60.html#IDX172">Contig Editor: annotations</A><LI><A HREF="gap5_20.html#IDX59">Contig Editor: annotations</A><LI><A HREF="gap4_63.html#IDX252">Contig Editor: Auto-diff traces</A><LI><A HREF="gap4_63.html#IDX247">Contig Editor: auto-display traces</A><LI><A HREF="gap4_63.html#IDX266">Contig Editor: auto-save</A><LI><A HREF="gap4_62.html#IDX226">Contig editor: break contig</A><LI><A HREF="gap4_62.html#IDX198">Contig Editor: commands menu</A><LI><A HREF="gap4_63.html#IDX263">Contig Editor: Compare Strands</A><LI><A HREF="gap4_77.html#IDX334">Contig editor: confidence values</A><LI><A HREF="gap4_52.html#IDX118">Contig Editor: cursor movement</A><LI><A HREF="gap4_55.html#IDX127">Contig Editor: cursor</A><LI><A HREF="gap5_15.html#IDX42">Contig Editor: cursor</A><LI><A HREF="gap4_57.html#IDX165">Contig Editor: cutoff data</A><LI><A HREF="gap5_16.html#IDX50">Contig Editor: cutoff data</A><LI><A HREF="gap4_57.html#IDX162">Contig Editor: cutoff values</A><LI><A HREF="gap5_16.html#IDX46">Contig Editor: cutoff values</A><LI><A HREF="gap4_64.html#IDX294">Contig editor: disassemble readings</A><LI><A HREF="gap4_62.html#IDX202">Contig Editor: Dump Contig</A><LI><A HREF="gap4_56.html#IDX152">Contig Editor: edit by base confidence</A><LI><A HREF="gap4_56.html#IDX151">Contig Editor: edit by base type</A><LI><A HREF="gap4_56.html#IDX158">Contig Editor: edit mode sets</A><LI><A HREF="gap4_56.html#IDX129">Contig Editor: edit modes</A><LI><A HREF="gap4_54.html#IDX125">Contig Editor: editing features</A><LI><A HREF="gap5_14.html#IDX40">Contig Editor: editing features</A><LI><A HREF="gap4_58.html#IDX167">Contig Editor: editing keys</A><LI><A HREF="gap5_17.html#IDX52">Contig Editor: editing keys</A><LI><A HREF="gap4_75.html#IDX330">Contig Editor: editing techniques</A><LI><A HREF="gap4_63.html#IDX257">Contig Editor: group readings</A><LI><A HREF="gap5_22.html#IDX71">Contig Editor: Group Readings</A><LI><A HREF="gap4_63.html#IDX260">Contig Editor: Highlight Disagreements</A><LI><A HREF="gap5_22.html#IDX73">Contig Editor: Highlight Disagreements</A><LI><A HREF="gap4_53.html#IDX120">Contig Editor: highlighting readings</A><LI><A HREF="gap5_13.html#IDX35">Contig Editor: highlighting readings</A><LI><A HREF="gap4_71.html#IDX312">Contig Editor: information line</A><LI><A HREF="gap5_25.html#IDX84">Contig Editor: information line</A><LI><A HREF="gap4_72.html#IDX324">Contig Editor: joining</A><LI><A HREF="gap5_26.html#IDX95">Contig Editor: joining</A><LI><A HREF="gap4_62.html#IDX216">Contig Editor: List Confidence</A><LI><A HREF="gap4_56.html#IDX157">Contig Editor: mode sets</A><LI><A HREF="gap4_73.html#IDX325">Contig Editor: multiple editors</A><LI><A HREF="gap5_27.html#IDX96">Contig Editor: multiple editors</A><LI><A HREF="gap4_62.html#IDX219">Contig Editor: mutation reporting</A><LI><A HREF="gap4_53.html#IDX119">Contig Editor: names display</A><LI><A HREF="gap5_13.html#IDX34">Contig Editor: names display</A><LI><A HREF="gap4_62.html#IDX222">Contig Editor: oligo selection</A><LI><A HREF="gap4_62.html#IDX223">Contig Editor: primer selection</A><LI><A HREF="gap4_65.html#IDX301">Contig Editor: Primer selection</A><LI><A HREF="gap5_23.html#IDX79">Contig Editor: Primer selection</A><LI><A HREF="gap4_57.html#IDX161">Contig Editor: quality values</A><LI><A HREF="gap5_16.html#IDX45">Contig Editor: quality values</A><LI><A HREF="gap4_74.html#IDX327">Contig Editor: quitting</A><LI><A HREF="gap5_28.html#IDX98">Contig Editor: quitting</A><LI><A HREF="gap4_67.html#IDX306">Contig Editor: Reference sequence</A><LI><A HREF="gap4_67.html#IDX307">Contig Editor: Reference traces</A><LI><A HREF="gap4_64.html#IDX295">Contig editor: remove reading</A><LI><A HREF="gap4_62.html#IDX212">Contig Editor: Save Consensus Trace</A><LI><A HREF="gap4_63.html#IDX235">Contig Editor: saving configuration</A><LI><A HREF="gap5_22.html#IDX69">Contig Editor: saving configuration</A><LI><A HREF="gap4_63.html#IDX234">Contig Editor: saving settings</A><LI><A HREF="gap5_22.html#IDX68">Contig Editor: saving settings</A><LI><A HREF="gap4_62.html#IDX207">Contig Editor: saving to file</A><LI><A HREF="gap4_62.html#IDX199">Contig Editor: saving</A><LI><A HREF="gap5_12.html#IDX33">Contig Editor: scrolling</A><LI><A HREF="gap4_61.html#IDX175">Contig Editor: searching</A><LI><A HREF="gap5_21.html#IDX62">Contig Editor: searching</A><LI><A HREF="gap5_19.html#IDX56">Contig Editor: selecting sequences</A><LI><A HREF="gap4_59.html#IDX169">Contig Editor: selections</A><LI><A HREF="gap5_18.html#IDX54">Contig Editor: selections</A><LI><A HREF="gap4_63.html#IDX283">Contig Editor: set active tags</A><LI><A HREF="gap4_63.html#IDX287">Contig Editor: set default confidences</A><LI><A HREF="gap4_63.html#IDX291">Contig Editor: Set or unset saving of undo</A><LI><A HREF="gap4_63.html#IDX285">Contig Editor: set output list</A><LI><A HREF="gap4_63.html#IDX230">Contig Editor: settings menu</A><LI><A HREF="gap5_22.html#IDX64">Contig Editor: settings menu</A><LI><A HREF="gap4_63.html#IDX274">Contig Editor: show consensus quality</A><LI><A HREF="gap4_63.html#IDX277">Contig Editor: show edits</A><LI><A HREF="gap4_63.html#IDX273">Contig Editor: show reading quality</A><LI><A HREF="gap4_63.html#IDX238">Contig Editor: Show Strands</A><LI><A HREF="gap4_63.html#IDX236">Contig Editor: status line</A><LI><A HREF="gap4_83.html#IDX338">Contig Editor: summary</A><LI><A HREF="gap5_29.html#IDX100">Contig Editor: summary</A><LI><A HREF="gap4_60.html#IDX173">Contig Editor: tags</A><LI><A HREF="gap5_20.html#IDX60">Contig Editor: tags</A><LI><A HREF="gap4_75.html#IDX331">Contig Editor: techniques</A><LI><A HREF="gap4_63.html#IDX281">Contig Editor: template names</A><LI><A HREF="gap4_70.html#IDX309">Contig Editor: template status</A><LI><A HREF="gap4_63.html#IDX265">Contig Editor: toggle auto-save</A><LI><A HREF="gap4_63.html#IDX244">Contig Editor: Trace Display menu</A><LI><A HREF="gap4_66.html#IDX303">Contig Editor: trace display</A><LI><A HREF="gap5_24.html#IDX81">Contig Editor: trace display</A><LI><A HREF="gap4_63.html#IDX241">Contig Editor: translations using feature tables</A><LI><A HREF="gap4_63.html#IDX240">Contig Editor: translations</A><LI><A HREF="gap4_99.html#IDX386">contig joining</A><LI><A HREF="gap5_42.html#IDX127">contig joining</A><LI><A HREF="gap4_157.html#IDX558">Contig names</A><LI><A HREF="gap4_22.html#IDX42">contig naming</A><LI><A HREF="gap5_4.html#IDX11">contig naming</A><LI><A HREF="gap4_130.html#IDX453">Contig navigation</A><LI><A HREF="gap4_169.html#IDX602">Contig order, reset: doctor database</A><LI><A HREF="gap4_21.html#IDX38">Contig order: Contig Selector</A><LI><A HREF="gap5_3.html#IDX7">Contig order: Contig Selector</A><LI><A HREF="gap4_130.html#IDX454">Contig region</A><LI><A HREF="gap4_23.html#IDX45">Contig Selector: changing the contig order</A><LI><A HREF="gap5_5.html#IDX14">Contig Selector: changing the contig order</A><LI><A HREF="gap4_21.html#IDX37">Contig Selector: Contig order</A><LI><A HREF="gap5_3.html#IDX6">Contig Selector: Contig order</A><LI><A HREF="gap4_24.html#IDX47">Contig Selector: menus</A><LI><A HREF="gap5_6.html#IDX16">Contig Selector: menus</A><LI><A HREF="gap4_23.html#IDX46">Contig Selector: saving the contig order</A><LI><A HREF="gap5_5.html#IDX15">Contig Selector: saving the contig order</A><LI><A HREF="gap4_22.html#IDX39">Contig Selector: selecting contigs</A><LI><A HREF="gap5_4.html#IDX8">Contig Selector: selecting contigs</A><LI><A HREF="gap4_163.html#IDX578">Contig structure: doctor database</A><LI><A HREF="gap4_168.html#IDX601">Contig, deletion of: doctor database</A><LI><A HREF="gap4_30.html#IDX74">Contig: template display</A><LI><A HREF="gap4_71.html#IDX317">CONTIG_BRIEF_FORMAT</A><LI><A HREF="gap5_25.html#IDX90">CONTIG_BRIEF_FORMAT</A><LI><A HREF="gap4_22.html#IDX43">contigs - identifying</A><LI><A HREF="gap5_4.html#IDX12">contigs - identifying</A><LI><A HREF="gap4_140.html#IDX479">contigs list</A><LI><A HREF="gap5_70.html#IDX191">contigs list</A><LI><A HREF="gap4_20.html#IDX36">Contigs marking</A><LI><A HREF="gap4_20.html#IDX34">Contigs masking</A><LI><A HREF="gap4_142.html#IDX497">Contigs to Readings: lists</A><LI><A HREF="gap5_72.html#IDX209">Contigs to Readings: lists</A><LI><A HREF="gap4_62.html#IDX203">Contigs: printing</A><LI><A HREF="gap4_62.html#IDX205">Contigs: saving to file</A><LI><A HREF="pregap4_66.html#IDX136">convert_trace.p4m</A><LI><A HREF="manpages_2.html#IDX1">convert_trace: man page</A><LI><A HREF="pregap4_19.html#IDX31">convert_trace</A><LI><A HREF="gap4_156.html#IDX546">Copy Database</A><LI><A HREF="gap4_141.html#IDX490">Copy list</A><LI><A HREF="gap5_71.html#IDX202">Copy list</A><LI><A HREF="gap4_159.html#IDX563">Copy reads: dialogue</A><LI><A HREF="gap4_158.html#IDX559">copy reads</A><LI><A HREF="spin_49.html#IDX168">Copy sequence: spin</A><LI><A HREF="manpages_3.html#IDX2">Copy_db: man page</A><LI><A HREF="manpages_4.html#IDX3">Copy_reads: man_page</A><LI><A HREF="formats_20.html#IDX53">CR: experiment file line type</A><LI><A HREF="gap4_141.html#IDX494">Create list</A><LI><A HREF="gap5_71.html#IDX206">Create list</A><LI><A HREF="pregap4_88.html#IDX318">create</A><LI><A HREF="pregap4_91.html#IDX342">create_dialogue</A><LI><A HREF="gap4_154.html#IDX537">Creating a new database</A><LI><A HREF="pregap4_72.html#IDX179">Cross_match configuration</A><LI><A HREF="pregap4_24.html#IDX41">Cross_match module</A><LI><A HREF="pregap4_72.html#IDX180">cross_match_svec.p4m</A><LI><A HREF="spin_46.html#IDX149">Crosshairs: spin</A><LI><A HREF="formats_20.html#IDX54">CS: experiment file line type</A><LI><A HREF="spin_43.html#IDX121">Cursor dragging:spin</A><LI><A HREF="spin_43.html#IDX120">Cursor linking:spin</A><LI><A HREF="spin_43.html#IDX119">Cursor positioning:spin</A><LI><A HREF="gap4_55.html#IDX126">Cursor: contig editor</A><LI><A HREF="gap5_15.html#IDX41">Cursor: contig editor</A><LI><A HREF="spin_46.html#IDX148">Cursor: spin</A><LI><A HREF="gap4_49.html#IDX113">Cut sites: restriction enzymes</A><LI><A HREF="gap4_57.html#IDX163">Cutoff data: contig editor</A><LI><A HREF="gap5_16.html#IDX48">Cutoff data: contig editor</A><LI><A HREF="trev_6.html#IDX18">Cutoff data: Trev</A><LI><A HREF="gap4_57.html#IDX160">Cutoff values: contig editor</A><LI><A HREF="gap5_16.html#IDX44">Cutoff values: contig editor</A><LI><A HREF="pregap4_81.html#IDX242">cutoff</A><LI><A HREF="spin_44.html#IDX141">Cutting sites: restriction enzymes:spin</A><LI><A HREF="formats_20.html#IDX55">CV: experiment file line type</A></DIR>
<A NAME="LETd"></A><H2>d</H2>
<DIR>
<LI><A HREF="gap4_17.html#IDX26">data hidden</A><LI><A HREF="gap5_2.html#IDX1">database creation</A><LI><A HREF="pregap4_53.html#IDX99">Database integration</A><LI><A HREF="gap4_4.html#IDX6">database limits</A><LI><A HREF="gap4_132.html#IDX462">Database merging</A><LI><A HREF="gap4_4.html#IDX4">database readonly access</A><LI><A HREF="gap4_132.html#IDX463">Database splitting</A><LI><A HREF="gap4_163.html#IDX574">Database structure: doctor database</A><LI><A HREF="gap4_4.html#IDX3">database write access</A><LI><A HREF="pregap4_54.html#IDX101">Database, plain text format</A><LI><A HREF="gap4_156.html#IDX544">Database: backups</A><LI><A HREF="gap4_152.html#IDX529">Database: busy file</A><LI><A HREF="gap4_154.html#IDX539">Database: creating new</A><LI><A HREF="gap4_152.html#IDX526">Database: gap4 filenames</A><LI><A HREF="gap4_4.html#IDX14">database: gap4 maxdb</A><LI><A HREF="gap4_4.html#IDX15">database: gap4 maxseq</A><LI><A HREF="gap4_152.html#IDX533">Database: locked</A><LI><A HREF="gap4_157.html#IDX557">Database: maximum size</A><LI><A HREF="gap4_154.html#IDX538">Database: new</A><LI><A HREF="gap4_155.html#IDX542">Database: opening</A><LI><A HREF="gap4_152.html#IDX532">Database: readonly</A><LI><A HREF="pregap4_88.html#IDX316">database_name</A><LI><A HREF="pregap4_88.html#IDX317">database_version</A><LI><A HREF="pregap4_71.html#IDX166">def_5_pos</A><LI><A HREF="gap4_166.html#IDX594">Delete annotations</A><LI><A HREF="gap4_168.html#IDX599">Delete contig: doctor database</A><LI><A HREF="gap5_53.html#IDX158">Delete Contigs</A><LI><A HREF="gap4_141.html#IDX495">Delete list</A><LI><A HREF="gap5_71.html#IDX207">Delete list</A><LI><A HREF="spin_49.html#IDX179">Delete sequence: spin</A><LI><A HREF="mutations_2.html#IDX2">detection of mutations: introduction</A><LI><A HREF="pregap4_74.html#IDX198">diag_score</A><LI><A HREF="gap4_134.html#IDX466">Difference clipping</A><LI><A HREF="pregap4_88.html#IDX320">difference_clip</A><LI><A HREF="spin_12.html#IDX4">Dinucleotide frequencies:spin</A><LI><A HREF="gap4_41.html#IDX95">Diploid Graph</A><LI><A HREF="gap4_90.html#IDX362">Directed assembly</A><LI><A HREF="interface_14.html#IDX31">Directories: file browser</A><LI><A HREF="gap4_4.html#IDX18">directories: trace files</A><LI><A HREF="gap4_153.html#IDX536">Directories</A><LI><A HREF="gap4_106.html#IDX407">Disassemble readings</A><LI><A HREF="gap5_52.html#IDX156">Disassemble readings</A><LI><A HREF="gap4_64.html#IDX297">Disassembly: contig editor</A><LI><A HREF="gap4_61.html#IDX194">Discrepancies: searching for in contig editor</A><LI><A HREF="spin_43.html#IDX122">Display interaction:spin</A><LI><A HREF="spin_44.html#IDX137">DNA character set</A><LI><A HREF="spin_16.html#IDX19">DNA translation:spin</A><LI><A HREF="gap4_163.html#IDX581">Doctor database: annotation structure</A><LI><A HREF="gap4_163.html#IDX585">Doctor database: clone structure</A><LI><A HREF="gap4_169.html#IDX603">Doctor database: contig order</A><LI><A HREF="gap4_163.html#IDX579">Doctor database: contig structure</A><LI><A HREF="gap4_163.html#IDX575">Doctor database: database structure</A><LI><A HREF="gap4_168.html#IDX600">Doctor database: delete contig</A><LI><A HREF="gap4_165.html#IDX592">Doctor database: extending structures</A><LI><A HREF="gap4_163.html#IDX588">Doctor database: note structure</A><LI><A HREF="gap4_163.html#IDX586">Doctor database: original clone structure</A><LI><A HREF="gap4_163.html#IDX577">Doctor database: reading structure</A><LI><A HREF="gap4_169.html#IDX604">Doctor database: reset contig order</A><LI><A HREF="gap4_167.html#IDX598">Doctor database: shift readings</A><LI><A HREF="gap4_163.html#IDX583">Doctor database: template structure</A><LI><A HREF="gap4_162.html#IDX573">Doctor Database</A><LI><A HREF="spin_45.html#IDX147">Dot plot: spin</A><LI><A HREF="gap4_63.html#IDX261">Dots: contig editor highlight disagreements</A><LI><A HREF="gap5_22.html#IDX74">Dots: contig editor highlight disagreements</A><LI><A HREF="gap4_108.html#IDX409">Double strand</A><LI><A HREF="spin_44.html#IDX144">Double stranded sequence listing:spin</A><LI><A HREF="formats_20.html#IDX56">DR: experiment file line type</A><LI><A HREF="spin_46.html#IDX151">Drag and drop graphics: spin</A><LI><A HREF="formats_20.html#IDX57">DT: experiment file line type</A><LI><A HREF="gap4_62.html#IDX201">Dump Contig: contig editor</A><LI><A HREF="spin_44.html#IDX142">Dumping results to file:spin</A><LI><A HREF="spin_40.html#IDX104">Duplicate matches: spin</A></DIR>
<A NAME="LETe"></A><H2>e</H2>
<DIR>
<LI><A HREF="pregap4_63.html#IDX128">eba.p4m</A><LI><A HREF="manpages_5.html#IDX4">eba: man page</A><LI><A HREF="gap4_56.html#IDX150">Edit by base confidence: contig editor</A><LI><A HREF="gap4_56.html#IDX149">Edit by base type: contig editor</A><LI><A HREF="gap4_141.html#IDX493">Edit list</A><LI><A HREF="gap5_71.html#IDX205">Edit list</A><LI><A HREF="gap4_56.html#IDX155">Edit mode sets: contig editor</A><LI><A HREF="gap4_56.html#IDX128">Edit modes: contig editor</A><LI><A HREF="gap4_149.html#IDX518">Edit notebooks</A><LI><A HREF="gap4_16.html#IDX24">Editing and base accuracies</A><LI><A HREF="gap4_77.html#IDX332">Editing techniques: confidence values</A><LI><A HREF="gap4_78.html#IDX335">Editing techniques: overcalls</A><LI><A HREF="gap4_75.html#IDX329">Editing techniques</A><LI><A HREF="trev_6.html#IDX22">Editing the sequence: Trev</A><LI><A HREF="gap4_54.html#IDX124">Editing: contig editor</A><LI><A HREF="gap5_14.html#IDX39">Editing: contig editor</A><LI><A HREF="trev_6.html#IDX16">Editing: Trev</A><LI><A HREF="pregap4_89.html#IDX323">Email configuration</A><LI><A HREF="pregap4_43.html#IDX81">Email module</A><LI><A HREF="pregap4_89.html#IDX324">email.p4m</A><LI><A HREF="pregap4_89.html#IDX327">email_address</A><LI><A HREF="pregap4_89.html#IDX326">email_args</A><LI><A HREF="pregap4_89.html#IDX325">email_progam</A><LI><A HREF="spin_49.html#IDX162">EMBOSS</A><LI><A HREF="formats_20.html#IDX58">EN: experiment file line type</A><LI><A HREF="pregap4_60.html#IDX120">enabled</A><LI><A HREF="pregap4_82.html#IDX253">end</A><LI><A HREF="pregap4_88.html#IDX314">Enter assembly configuration</A><LI><A HREF="pregap4_42.html#IDX79">Enter assembly module</A><LI><A HREF="pregap4_83.html#IDX266">enter_all</A><LI><A HREF="pregap4_88.html#IDX315">enter_assembly.p4m</A><LI><A HREF="gap4_127.html#IDX447">entering annotations from file</A><LI><A HREF="gap4_152.html#IDX527">Entering readings</A><LI><A HREF="gap5_35.html#IDX111">Entering readings</A><LI><A HREF="gap4_127.html#IDX445">entering tags from file</A><LI><A HREF="interface_7.html#IDX9">Entry boxes</A><LI><A HREF="spin_49.html#IDX158">Entry sequence: spin</A><LI><A HREF="vector_clip_15.html#IDX29">error codes in screen_seq</A><LI><A HREF="vector_clip_6.html#IDX5">error codes in vector_clip</A><LI><A HREF="gap4_100.html#IDX394">error messages: find internal joins</A><LI><A HREF="gap5_43.html#IDX135">error messages: find internal joins</A><LI><A HREF="gap4_100.html#IDX395">error messages: maxseq</A><LI><A HREF="gap5_43.html#IDX136">error messages: maxseq</A><LI><A HREF="interface_9.html#IDX19">Error window</A><LI><A HREF="pregap4_63.html#IDX127">Estimate base accuracies configuration</A><LI><A HREF="pregap4_16.html#IDX24">Estimate base accuracies module</A><LI><A HREF="gap4_61.html#IDX189">Evidence for Edit1: contig editor</A><LI><A HREF="gap4_61.html#IDX192">Evidence for Edit2: contig editor</A><LI><A HREF="formats_20.html#IDX59">EX: experiment file line type</A><LI><A HREF="pregap4_91.html#IDX357">Example code</A><LI><A HREF="formats_21.html#IDX95">Example experiment file</A><LI><A HREF="spin_40.html#IDX100">Expected number of matches in spin</A><LI><A HREF="pregap4_55.html#IDX103">Experiment File line types</A><LI><A HREF="gap4_157.html#IDX552">Experiment file name length restrictions</A><LI><A HREF="gap4_157.html#IDX551">Experiment file name restrictions</A><LI><A HREF="formats_2.html#IDX4">Experiment file name restrictions</A><LI><A HREF="formats_21.html#IDX94">Experiment file: example</A><LI><A HREF="formats_20.html#IDX42">Experiment file: explanation of records</A><LI><A HREF="formats_22.html#IDX96">Experiment file: unsupported additions</A><LI><A HREF="formats_19.html#IDX40">Experiment files: record types</A><LI><A HREF="formats_18.html#IDX39">Experiment files</A><LI><A HREF="gap5_40.html#IDX122">Export GFF</A><LI><A HREF="gap5_41.html#IDX124">Export Sequences</A><LI><A HREF="gap5_40.html#IDX121">Export Tags</A><LI><A HREF="gap4_115.html#IDX426">Extended consensus</A><LI><A HREF="gap4_165.html#IDX591">Extending structures: doctor database</A><LI><A HREF="gap4_176.html#IDX622">Extension penalty for alignments</A><LI><A HREF="pregap4_79.html#IDX224">Extract sequence configuration</A><LI><A HREF="pregap4_30.html#IDX53">Extract Sequence module</A><LI><A HREF="manpages_7.html#IDX6">extract_fastq: man page</A><LI><A HREF="pregap4_79.html#IDX225">extract_seq.p4m</A><LI><A HREF="manpages_6.html#IDX5">extract_seq: man page</A><LI><A HREF="vector_clip_11.html#IDX24">extraneous readings: filtering out</A></DIR>
<A NAME="LETf"></A><H2>f</H2>
<DIR>
<LI><A HREF="pregap4_86.html#IDX282">FakII assembly configuration</A><LI><A HREF="pregap4_40.html#IDX75">FakII assembly module</A><LI><A HREF="pregap4_86.html#IDX283">fakii_assemble.p4m</A><LI><A HREF="pregap4_4.html#IDX3">FASTA files: pregap4</A><LI><A HREF="gap4_114.html#IDX424">Fasta output from Gap</A><LI><A HREF="gap5_58.html#IDX169">Fasta output from Gap</A><LI><A HREF="gap4_63.html#IDX242">Feature tables: Translation in Contig Editor</A><LI><A HREF="spin_49.html#IDX156">feature tables</A><LI><A HREF="interface_14.html#IDX33">File browser: directories</A><LI><A HREF="interface_14.html#IDX34">File browser: files</A><LI><A HREF="interface_14.html#IDX35">File browser: filters</A><LI><A HREF="interface_14.html#IDX37">File browser: formats</A><LI><A HREF="interface_13.html#IDX30">File browser: introduction</A><LI><A HREF="interface_12.html#IDX29">File browser</A><LI><A HREF="vector_clip_9.html#IDX14">file formats for vectors</A><LI><A HREF="gap4_24.html#IDX48">File menu: Contig Selector</A><LI><A HREF="gap5_6.html#IDX17">File menu: Contig Selector</A><LI><A HREF="gap4_157.html#IDX550">File name restrictions</A><LI><A HREF="formats_2.html#IDX2">File name restrictions</A><LI><A HREF="gap4_142.html#IDX498">File of filenames generation</A><LI><A HREF="gap5_72.html#IDX210">File of filenames generation</A><LI><A HREF="formats_9.html#IDX29">File structure: SCF</A><LI><A HREF="pregap4_91.html#IDX354">file_error</A><LI><A HREF="pregap4_91.html#IDX355">file_id</A><LI><A HREF="pregap4_91.html#IDX353">file_type</A><LI><A HREF="trev_2.html#IDX6">Filebrowser: Trev</A><LI><A HREF="pregap4_11.html#IDX9">Files, specifying</A><LI><A HREF="interface_14.html#IDX32">Files: file browser</A><LI><A HREF="vector_clip_11.html#IDX23">filtering out extraneous readings</A><LI><A HREF="interface_14.html#IDX36">Filters: file browser</A><LI><A HREF="spin_37.html#IDX91">Find best diagonals: spin</A><LI><A HREF="gap4_100.html#IDX393">Find internal joins: dialogue</A><LI><A HREF="gap5_43.html#IDX134">Find internal joins: dialogue</A><LI><A HREF="gap4_99.html#IDX384">Find internal joins</A><LI><A HREF="gap5_42.html#IDX125">Find internal joins</A><LI><A HREF="spin_36.html#IDX90">Find matching words: spin</A><LI><A HREF="gap4_131.html#IDX455">Find oligos</A><LI><A HREF="gap5_47.html#IDX143">Find oligos</A><LI><A HREF="spin_17.html#IDX21">Find open reading frames:spin</A><LI><A HREF="gap4_97.html#IDX378">Find read pairs: display</A><LI><A HREF="gap5_46.html#IDX142">Find read pairs: display</A><LI><A HREF="gap4_98.html#IDX380">Find read pairs: example</A><LI><A HREF="gap4_98.html#IDX379">Find read pairs: output</A><LI><A HREF="gap4_98.html#IDX383">Find read pairs: reading lines</A><LI><A HREF="gap4_98.html#IDX381">Find read pairs: template lines</A><LI><A HREF="gap4_96.html#IDX376">Find read pairs</A><LI><A HREF="gap5_45.html#IDX140">Find read pairs</A><LI><A HREF="gap4_101.html#IDX397">Find repeats</A><LI><A HREF="gap5_44.html#IDX138">Find repeats</A><LI><A HREF="gap4_131.html#IDX457">Find sequences</A><LI><A HREF="gap5_47.html#IDX145">Find sequences</A><LI><A HREF="spin_35.html#IDX89">Find similar spans: spin</A><LI><A HREF="manpages_8.html#IDX7">find_renz: man page</A><LI><A HREF="spin_25.html#IDX45">Finding genes: Introduction:spin</A><LI><A HREF="gap4_99.html#IDX390">finding joins</A><LI><A HREF="gap5_42.html#IDX131">finding joins</A><LI><A HREF="gap4_99.html#IDX389">finding overlaps</A><LI><A HREF="gap5_42.html#IDX130">finding overlaps</A><LI><A HREF="spin_31.html#IDX71">Finding protein genes:spin</A><LI><A HREF="spin_44.html#IDX129">Finding strings:spin</A><LI><A HREF="formats_20.html#IDX60">FM: experiment file line type</A><LI><A HREF="gap4_173.html#IDX613">fonts, adjusting</A><LI><A HREF="trev_3.html#IDX10">Fonts, within trev</A><LI><A HREF="pregap4_13.html#IDX16">Fonts</A><LI><A HREF="interface_15.html#IDX39">Fonts</A><LI><A HREF="spin_40.html#IDX106">Format of protein score matrix</A><LI><A HREF="formats_25.html#IDX100">format: vector sequences</A><LI><A HREF="vector_clip_8.html#IDX8">format: vector_primer files</A><LI><A HREF="formats_24.html#IDX99">format: vector_primer files</A><LI><A HREF="interface_14.html#IDX38">Formats: file browser</A><LI><A HREF="vector_clip_9.html#IDX15">formats: vector files</A><LI><A HREF="pregap4_91.html#IDX356">Functions, builtin</A><LI><A HREF="pregap4_91.html#IDX337">Functions, in modules</A></DIR>
<A NAME="LETg"></A><H2>g</H2>
<DIR>
<LI><A HREF="gap4_176.html#IDX623">Gap penalties for alignments</A><LI><A HREF="gap4_84.html#IDX349">gap4 assembly limits</A><LI><A HREF="gap4_4.html#IDX7">gap4 database limits: resetting</A><LI><A HREF="gap4_4.html#IDX9">gap4 database sizes: resetting</A><LI><A HREF="gap4_4.html#IDX8">gap4 database sizes</A><LI><A HREF="gap4_4.html#IDX10">gap4 database: maxdb</A><LI><A HREF="gap4_4.html#IDX11">gap4 database: maxseq</A><LI><A HREF="gap4_4.html#IDX12">gap4 database: reading length limits</A><LI><A HREF="gap4_84.html#IDX351">gap4 database: resetting sizes</A><LI><A HREF="pregap4_83.html#IDX258">Gap4 shotgun assembly configuration</A><LI><A HREF="pregap4_37.html#IDX69">Gap4 shotgun assembly module</A><LI><A HREF="gap4_84.html#IDX350">gap4: resetting assembly limits</A><LI><A HREF="gap4_4.html#IDX19">gap4: viewing trace files</A><LI><A HREF="gap4_3.html#IDX1">Gap4</A><LI><A HREF="pregap4_83.html#IDX259">gap4_assemble.p4m</A><LI><A HREF="gap4_170.html#IDX606">gaprc</A><LI><A HREF="spin_25.html#IDX46">Gene finding: Introduction:spin</A><LI><A HREF="pregap4_61.html#IDX121">General configuration configuration</A><LI><A HREF="pregap4_15.html#IDX22">General configuration module</A><LI><A HREF="pregap4_86.html#IDX291">generate_constraints</A><LI><A HREF="spin_15.html#IDX17">Genetic code:spin</A><LI><A HREF="gap4_175.html#IDX618">Genetic code</A><LI><A HREF="spin_49.html#IDX160">Get sequence: spin</A><LI><A HREF="manpages_10.html#IDX9">get_comment: man page</A><LI><A HREF="manpages_11.html#IDX10">get_scf_field: man page</A><LI><A HREF="manpages_9.html#IDX8">getABIfield: man page</A><LI><A HREF="gap5_40.html#IDX123">GFF: exporting</A><LI><A HREF="gap5_39.html#IDX119">GFF: importing from</A><LI><A HREF="pregap4_91.html#IDX351">Global variables</A><LI><A HREF="trev_8.html#IDX31">Goto file button, trev</A><LI><A HREF="pregap4_86.html#IDX286">graph_d_limit</A><LI><A HREF="pregap4_86.html#IDX284">graph_e_limit</A><LI><A HREF="pregap4_86.html#IDX285">graph_o_threshold</A><LI><A HREF="spin_46.html#IDX152">Graphics rearrangement: spin</A><LI><A HREF="interface_10.html#IDX24">Graphics windows: user interface</A><LI><A HREF="gap4_63.html#IDX258">Group Readings: contig editor</A><LI><A HREF="gap5_22.html#IDX70">Group Readings: contig editor</A><LI><A HREF="gap4_179.html#IDX633">GTAGDB</A></DIR>
<A NAME="LETh"></A><H2>h</H2>
<DIR>
<LI><A HREF="gap4_42.html#IDX97">Haplotype assignment</A><LI><A HREF="gap4_177.html#IDX629">HASH= RAWDATA accessor</A><LI><A HREF="manpages_12.html#IDX11">hash_exp: man page</A><LI><A HREF="manpages_13.html#IDX12">hash_extract: man page</A><LI><A HREF="manpages_14.html#IDX13">hash_list: man page</A><LI><A HREF="manpages_15.html#IDX14">hash_tar: man page</A><LI><A HREF="formats_4.html#IDX12">Header record: SCF</A><LI><A HREF="formats_4.html#IDX13">Header: SCF structure</A><LI><A HREF="gap4_57.html#IDX164">Hidden data: contig editor</A><LI><A HREF="gap5_16.html#IDX49">Hidden data: contig editor</A><LI><A HREF="gap4_99.html#IDX387">hidden data</A><LI><A HREF="gap4_17.html#IDX25">Hidden data</A><LI><A HREF="gap5_42.html#IDX128">hidden data</A><LI><A HREF="read_clipping_2.html#IDX3">hidden data</A><LI><A HREF="pregap4_91.html#IDX349">hidden</A><LI><A HREF="spin_40.html#IDX103">Hide duplicate matches: spin</A><LI><A HREF="gap4_26.html#IDX55">Hide, in Contig Comparator</A><LI><A HREF="gap5_8.html#IDX24">Hide, in Contig Comparator</A><LI><A HREF="gap4_63.html#IDX259">Highlight Disagreements: contig editor</A><LI><A HREF="gap5_22.html#IDX72">Highlight Disagreements: contig editor</A><LI><A HREF="gap4_144.html#IDX504">Highlight readings list</A><LI><A HREF="gap4_53.html#IDX121">Highlighting readings in the editor</A><LI><A HREF="gap5_13.html#IDX36">Highlighting readings in the editor</A></DIR>
<A NAME="LETi"></A><H2>i</H2>
<DIR>
<LI><A HREF="formats_20.html#IDX61">ID: experiment file line type</A><LI><A HREF="gap4_22.html#IDX44">identifying contigs</A><LI><A HREF="gap5_4.html#IDX13">identifying contigs</A><LI><A HREF="gap4_164.html#IDX590">Ignore check database</A><LI><A HREF="gap4_30.html#IDX69">Ignore single templates: template display</A><LI><A HREF="gap5_39.html#IDX120">Import GFF Annotations</A><LI><A HREF="pregap4_52.html#IDX97">Include config component</A><LI><A HREF="gap4_151.html#IDX525">INFO note type</A><LI><A HREF="gap4_71.html#IDX320">Information line: bases in contig editor</A><LI><A HREF="gap4_71.html#IDX310">Information line: contig editor</A><LI><A HREF="gap5_25.html#IDX82">Information line: contig editor</A><LI><A HREF="gap4_71.html#IDX316">Information line: contig in contig editor</A><LI><A HREF="gap5_25.html#IDX89">Information line: contig in contig editor</A><LI><A HREF="gap4_71.html#IDX314">Information line: readings in contig editor</A><LI><A HREF="gap5_25.html#IDX86">Information line: readings in contig editor</A><LI><A HREF="gap4_71.html#IDX318">Information line: tags in contig editor</A><LI><A HREF="gap5_25.html#IDX92">Information line: tags in contig editor</A><LI><A HREF="pregap4_53.html#IDX98">Information sources</A><LI><A HREF="gap4_26.html#IDX54">Information, in Contig Comparator</A><LI><A HREF="gap5_8.html#IDX23">Information, in Contig Comparator</A><LI><A HREF="trev_5.html#IDX14">Information: Trev</A><LI><A HREF="pregap4_61.html#IDX122">init.p4m</A><LI><A HREF="pregap4_91.html#IDX339">init</A><LI><A HREF="pregap4_67.html#IDX138">init_exp.p4m</A><LI><A HREF="manpages_16.html#IDX15">init_exp: man page</A><LI><A HREF="pregap4_67.html#IDX137">Initialise Experiment Files configuration</A><LI><A HREF="pregap4_20.html#IDX33">Initialise Experiment files module</A><LI><A HREF="gap5_67.html#IDX183">Insert Sizes</A><LI><A HREF="spin_43.html#IDX123">Interaction of displays:spin</A><LI><A HREF="pregap4_78.html#IDX222">Interactive clipping configuration</A><LI><A HREF="pregap4_29.html#IDX51">Interactive clipping module</A><LI><A HREF="pregap4_78.html#IDX223">interactive_clip.p4m</A><LI><A HREF="spin_49.html#IDX174">Interconvert t and u: spin</A><LI><A HREF="pregap4_3.html#IDX1">Introduction</A><LI><A HREF="spin_33.html#IDX86">Intron in tRNA:spin</A><LI><A HREF="spin_32.html#IDX75">Intron/exon boundaries:spin</A><LI><A HREF="gap4_26.html#IDX56">Invoke contig editors, in Contig Comparator</A><LI><A HREF="gap5_8.html#IDX25">Invoke contig editors, in Contig Comparator</A><LI><A HREF="gap4_26.html#IDX57">Invoke contig join editors, in Contig Comparator</A><LI><A HREF="gap5_8.html#IDX26">Invoke contig join editors, in Contig Comparator</A><LI><A HREF="gap4_26.html#IDX58">Invoke template display, in Contig Comparator</A><LI><A HREF="gap4_118.html#IDX435">IUB codes: consensus</A><LI><A HREF="gap5_59.html#IDX173">IUB codes: consensus</A><LI><A HREF="spin_44.html#IDX136">IUB symbols:spin</A></DIR>
<A NAME="LETj"></A><H2>j</H2>
<DIR>
<LI><A HREF="gap4_72.html#IDX323">Join Editor</A><LI><A HREF="gap5_26.html#IDX94">Join Editor</A><LI><A HREF="gap4_99.html#IDX385">joining contigs</A><LI><A HREF="gap5_42.html#IDX126">joining contigs</A></DIR>
<A NAME="LETk"></A><H2>k</H2>
<DIR>
<LI><A HREF="gap4_83.html#IDX339">Keyboard summary (contig editor)</A><LI><A HREF="gap5_29.html#IDX101">Keyboard summary (contig editor)</A></DIR>
<A NAME="LETl"></A><H2>l</H2>
<DIR>
<LI><A HREF="gap4_18.html#IDX30">Labelling contigs</A><LI><A HREF="gap4_18.html#IDX29">Labelling readings</A><LI><A HREF="formats_20.html#IDX62">LE: experiment file line type</A><LI><A HREF="interface_8.html#IDX14">Left mouse button: overview</A><LI><A HREF="pregap4_70.html#IDX163">left_num_uncalled</A><LI><A HREF="pregap4_70.html#IDX162">left_win_length</A><LI><A HREF="formats_20.html#IDX63">LI: experiment file line type</A><LI><A HREF="pregap4_81.html#IDX241">library</A><LI><A HREF="pregap4_55.html#IDX104">Line types, in experiment file</A><LI><A HREF="gap4_124.html#IDX443">List base confidence</A><LI><A HREF="gap5_65.html#IDX181">List base confidence</A><LI><A HREF="gap4_62.html#IDX215">List confidence: contig editor</A><LI><A HREF="gap4_123.html#IDX441">List confidence</A><LI><A HREF="gap5_64.html#IDX179">List confidence</A><LI><A HREF="gap5_67.html#IDX182">List Libraries</A><LI><A HREF="gap4_95.html#IDX372">Listbox</A><LI><A HREF="gap4_141.html#IDX483">Lists: commands</A><LI><A HREF="gap5_71.html#IDX195">Lists: commands</A><LI><A HREF="gap4_142.html#IDX496">Lists: Contigs to Readings</A><LI><A HREF="gap5_72.html#IDX208">Lists: Contigs to Readings</A><LI><A HREF="gap4_141.html#IDX484">Lists: copy</A><LI><A HREF="gap5_71.html#IDX196">Lists: copy</A><LI><A HREF="gap4_141.html#IDX488">Lists: create</A><LI><A HREF="gap5_71.html#IDX200">Lists: create</A><LI><A HREF="gap4_141.html#IDX489">Lists: delete</A><LI><A HREF="gap5_71.html#IDX201">Lists: delete</A><LI><A HREF="gap4_141.html#IDX487">Lists: edit</A><LI><A HREF="gap5_71.html#IDX199">Lists: edit</A><LI><A HREF="gap4_144.html#IDX503">Lists: highlight readings list</A><LI><A HREF="gap4_141.html#IDX485">Lists: load</A><LI><A HREF="gap5_71.html#IDX197">Lists: load</A><LI><A HREF="gap4_143.html#IDX499">Lists: minimal coverage</A><LI><A HREF="gap4_141.html#IDX486">Lists: save</A><LI><A HREF="gap5_71.html#IDX198">Lists: save</A><LI><A HREF="gap4_147.html#IDX512">Lists: Search annotation contents</A><LI><A HREF="gap4_145.html#IDX505">Lists: search sequence names</A><LI><A HREF="gap5_73.html#IDX211">Lists: search sequence names</A><LI><A HREF="gap4_146.html#IDX509">Lists: Search template names</A><LI><A HREF="gap4_140.html#IDX478">Lists: special names</A><LI><A HREF="gap5_70.html#IDX190">Lists: special names</A><LI><A HREF="gap4_144.html#IDX501">Lists: unattached readings</A><LI><A HREF="gap4_139.html#IDX477">Lists</A><LI><A HREF="gap5_69.html#IDX189">Lists</A><LI><A HREF="formats_20.html#IDX64">LN: experiment file line type</A><LI><A HREF="gap4_141.html#IDX491">Load list</A><LI><A HREF="gap5_71.html#IDX203">Load list</A><LI><A HREF="pregap4_47.html#IDX87">Load naming scheme</A><LI><A HREF="spin_49.html#IDX159">Load sequence: spin</A><LI><A HREF="spin_39.html#IDX93">Local alignment: spin</A><LI><A HREF="gap4_152.html#IDX534">locked database</A><LI><A HREF="gap4_110.html#IDX413">Long readings: suggestion of</A><LI><A HREF="pregap4_57.html#IDX112">low level pregap4 configuration</A><LI><A HREF="formats_20.html#IDX65">LT: experiment file line type</A></DIR>
<A NAME="LETm"></A><H2>m</H2>
<DIR>
<LI><A HREF="formats_4.html#IDX15">Magic number: SCF</A><LI><A HREF="manpages_18.html#IDX17">Make_weights: man page</A><LI><A HREF="manpages_17.html#IDX16">makeSCF: man page</A><LI><A HREF="pregap4_91.html#IDX348">mandatory</A><LI><A HREF="gap4_20.html#IDX35">Marking contigs</A><LI><A HREF="gap4_99.html#IDX392">marking</A><LI><A HREF="gap5_42.html#IDX133">marking</A><LI><A HREF="gap4_20.html#IDX33">Masking contigs</A><LI><A HREF="gap4_99.html#IDX391">masking</A><LI><A HREF="gap5_42.html#IDX132">masking</A><LI><A HREF="spin_40.html#IDX97">Match probabilities in spin</A><LI><A HREF="pregap4_77.html#IDX220">match_fraction</A><LI><A HREF="spin_44.html#IDX128">Matching strings:spin</A><LI><A HREF="gap4_176.html#IDX624">Matrix for alignments</A><LI><A HREF="pregap4_70.html#IDX158">max_extent</A><LI><A HREF="pregap4_76.html#IDX213">max_length</A><LI><A HREF="pregap4_83.html#IDX264">max_pads</A><LI><A HREF="pregap4_83.html#IDX265">max_pmismatch</A><LI><A HREF="gap4_181.html#IDX643">maxdb (command line option)</A><LI><A HREF="gap4_84.html#IDX348">maxdb: gap4 assembly</A><LI><A HREF="gap4_172.html#IDX611">maximum sequence length</A><LI><A HREF="gap4_181.html#IDX645">maxseq (command line option)</A><LI><A HREF="gap4_100.html#IDX396">maxseq: find internal joins</A><LI><A HREF="gap5_43.html#IDX137">maxseq: find internal joins</A><LI><A HREF="gap4_84.html#IDX347">maxseq: gap4 assembly</A><LI><A HREF="gap4_172.html#IDX610">maxseq</A><LI><A HREF="formats_20.html#IDX66">MC: experiment file line type</A><LI><A HREF="spin_48.html#IDX155">Memory saving: spin</A><LI><A HREF="spin_13.html#IDX10">Memory saving:spin</A><LI><A HREF="spin_13.html#IDX11">Memory usage:spin</A><LI><A HREF="gap4_174.html#IDX615">Menus, configuring</A><LI><A HREF="interface_5.html#IDX5">Menus</A><LI><A HREF="gap4_132.html#IDX460">Merging databases</A><LI><A HREF="interface_8.html#IDX15">Middle mouse button: overview</A><LI><A HREF="pregap4_71.html#IDX169">min_3_match</A><LI><A HREF="pregap4_71.html#IDX168">min_5_match</A><LI><A HREF="pregap4_70.html#IDX157">min_extent</A><LI><A HREF="pregap4_70.html#IDX159">min_length</A><LI><A HREF="pregap4_83.html#IDX263">min_match</A><LI><A HREF="gap4_143.html#IDX500">minimal coverage: lists</A><LI><A HREF="pregap4_87.html#IDX304">minmatch</A><LI><A HREF="pregap4_87.html#IDX305">minscore</A><LI><A HREF="formats_20.html#IDX67">MN: experiment file line type</A><LI><A HREF="gap4_56.html#IDX156">Mode sets: contig editor</A><LI><A HREF="pregap4_91.html#IDX336">Module functions</A><LI><A HREF="pregap4_91.html#IDX346">Module variables</A><LI><A HREF="pregap4_91.html#IDX358">Module, example code</A><LI><A HREF="pregap4_58.html#IDX115">MODULE_PATH</A><LI><A HREF="pregap4_56.html#IDX108">Modules, adding and removing</A><LI><A HREF="pregap4_60.html#IDX118">Modules, configuring</A><LI><A HREF="pregap4_91.html#IDX334">Modules, creating</A><LI><A HREF="pregap4_91.html#IDX335">Modules, overview</A><LI><A HREF="pregap4_58.html#IDX116">MODULES</A><LI><A HREF="spin_44.html#IDX134">Motif searching: percentage matches:spin</A><LI><A HREF="spin_24.html#IDX41">Motif searching:spin</A><LI><A HREF="spin_25.html#IDX50">motifs: spin</A><LI><A HREF="interface_8.html#IDX12">Mouse buttons: overview</A><LI><A HREF="interface_8.html#IDX11">Mouse control: overview</A><LI><A HREF="formats_20.html#IDX68">MT: experiment file line type</A><LI><A HREF="trev_8.html#IDX27">Multiple files in Trev</A><LI><A HREF="pregap4_82.html#IDX249">Mutation detection configuration</A><LI><A HREF="pregap4_33.html#IDX59">Mutation detection module</A><LI><A HREF="pregap4_48.html#IDX89">Mutation detection naming scheme</A><LI><A HREF="mutations_2.html#IDX1">Mutation detection: introduction</A><LI><A HREF="mutations_4.html#IDX5">mutation detection: reference sequences</A><LI><A HREF="mutations_4.html#IDX7">mutation detection: reference traces</A><LI><A HREF="mutations_2.html#IDX3">mutation report</A><LI><A HREF="gap4_62.html#IDX218">Mutation reporting: contig editor</A><LI><A HREF="pregap4_36.html#IDX68">Mutation scanner module</A><LI><A HREF="pregap4_48.html#IDX90">mutation_detection.p4t</A></DIR>
<A NAME="LETn"></A><H2>n</H2>
<DIR>
<LI><A HREF="gap4_137.html#IDX472">N-base clipping</A><LI><A HREF="pregap4_91.html#IDX338">name</A><LI><A HREF="gap4_53.html#IDX122">names in the editor</A><LI><A HREF="gap5_13.html#IDX37">names in the editor</A><LI><A HREF="gap4_22.html#IDX41">naming contigs</A><LI><A HREF="gap5_4.html#IDX10">naming contigs</A><LI><A HREF="pregap4_51.html#IDX95">Naming schemes, creating.</A><LI><A HREF="pregap4_34.html#IDX66">naming schemes: mutation detection</A><LI><A HREF="pregap4_34.html#IDX65">naming schemes: pregap4</A><LI><A HREF="pregap4_47.html#IDX88">Naming schemes</A><LI><A HREF="spin_44.html#IDX135">NC-IUB symbols:spin</A><LI><A HREF="gap4_154.html#IDX540">New database creation</A><LI><A HREF="gap4_27.html#IDX62">Next button, in Contig comparator</A><LI><A HREF="gap5_9.html#IDX30">Next button, in Contig comparator</A><LI><A HREF="trev_8.html#IDX29">Next button, trev</A><LI><A HREF="pregap4_81.html#IDX240">no_low_complexity</A><LI><A HREF="pregap4_81.html#IDX239">no_primate_rodent</A><LI><A HREF="gap4_181.html#IDX650">nocheck (command line option)</A><LI><A HREF="gap4_114.html#IDX423">Normal consensus</A><LI><A HREF="gap5_58.html#IDX168">Normal consensus</A><LI><A HREF="spin_30.html#IDX67">Normalisation: codon usage tables:spin</A><LI><A HREF="gap4_163.html#IDX587">Note structure: doctor database</A><LI><A HREF="gap4_150.html#IDX519">Notes: editing</A><LI><A HREF="gap4_149.html#IDX517">Notes: selecting</A><LI><A HREF="gap4_151.html#IDX520">Notes: special types</A><LI><A HREF="gap4_148.html#IDX516">Notes</A><LI><A HREF="spin_44.html#IDX138">Nucleotide symbols</A><LI><A HREF="pregap4_74.html#IDX197">num_diags</A></DIR>
<A NAME="LETo"></A><H2>o</H2>
<DIR>
<LI><A HREF="pregap4_70.html#IDX156">offset</A><LI><A HREF="pregap4_74.html#IDX193">Old cloning vector clip configuration</A><LI><A HREF="pregap4_74.html#IDX195">old_cloning_vector_clip.p4m</A><LI><A HREF="gap4_131.html#IDX456">Oligo search</A><LI><A HREF="gap5_47.html#IDX144">Oligo search</A><LI><A HREF="spin_44.html#IDX133">Oligo searching:spin</A><LI><A HREF="gap4_65.html#IDX300">Oligo selection: contig editor</A><LI><A HREF="gap5_23.html#IDX78">Oligo selection: contig editor</A><LI><A HREF="gap4_112.html#IDX417">Oligos: choosing for probes</A><LI><A HREF="formats_20.html#IDX69">ON: experiment file line type</A><LI><A HREF="formats_20.html#IDX70">OP: experiment file line type</A><LI><A HREF="gap5_2.html#IDX3">Open database</A><LI><A HREF="gap4_151.html#IDX522">OPEN note type</A><LI><A HREF="gap4_176.html#IDX621">Open penalty for alignments</A><LI><A HREF="spin_27.html#IDX55">Open reading frames:spin</A><LI><A HREF="gap4_155.html#IDX541">Opening databases</A><LI><A HREF="trev_2.html#IDX4">Opening trace files: Trev</A><LI><A HREF="gap4_95.html#IDX368">Ordering contigs:gap4</A><LI><A HREF="pregap4_87.html#IDX306">other_args</A><LI><A HREF="gap4_166.html#IDX593">Output annotations to file</A><LI><A HREF="gap4_50.html#IDX116">Output enzyme by enzyme: restriction enzymes plot</A><LI><A HREF="gap5_34.html#IDX108">Output enzyme by enzyme: restriction enzymes plot</A><LI><A HREF="gap4_50.html#IDX117">Output ordered on position: restriction enzymes plot</A><LI><A HREF="gap5_34.html#IDX109">Output ordered on position: restriction enzymes plot</A><LI><A HREF="interface_9.html#IDX18">Output window</A><LI><A HREF="gap4_78.html#IDX336">Overcalls: editing techniques</A><LI><A HREF="gap4_99.html#IDX388">overlap finding</A><LI><A HREF="gap5_42.html#IDX129">overlap finding</A></DIR>
<A NAME="LETp"></A><H2>p</H2>
<DIR>
<LI><A HREF="gap4_128.html#IDX448">pads: realigning</A><LI><A HREF="formats_20.html#IDX71">PC: experiment file line type</A><LI><A HREF="formats_20.html#IDX72">PD: primer data - the sequence of a primer</A><LI><A HREF="spin_44.html#IDX131">Percentage matches:spin</A><LI><A HREF="spin_13.html#IDX9">Persistence of results:spin</A><LI><A HREF="spin_49.html#IDX163">Personal search: spin</A><LI><A HREF="pregap4_87.html#IDX302">Phrap assembly configuration</A><LI><A HREF="pregap4_41.html#IDX77">Phrap assembly module</A><LI><A HREF="pregap4_87.html#IDX303">phrap_assemble.p4m</A><LI><A HREF="pregap4_64.html#IDX129">Phred configuration</A><LI><A HREF="pregap4_17.html#IDX26">Phred module</A><LI><A HREF="pregap4_64.html#IDX132">phred.log</A><LI><A HREF="pregap4_64.html#IDX130">phred.p4m</A><LI><A HREF="formats_25.html#IDX102">plain text</A><LI><A HREF="gap4_44.html#IDX102">Plot stop codons: examining the plot</A><LI><A HREF="gap4_44.html#IDX104">Plot stop codons: updating the plot</A><LI><A HREF="gap4_44.html#IDX101">Plot stop codons</A><LI><A HREF="spin_13.html#IDX6">Plotting base composition:spin</A><LI><A HREF="formats_20.html#IDX73">PN: experiment file line type</A><LI><A HREF="manpages_19.html#IDX19">polyA clipping</A><LI><A HREF="manpages_19.html#IDX18">polyA_clip: man page</A><LI><A HREF="manpages_19.html#IDX20">polyT clipping</A><LI><A HREF="formats_20.html#IDX74">PR: experiment file line type</A><LI><A HREF="pregap4_4.html#IDX4">pregap4: FASTA files</A><LI><A HREF="pregap4_34.html#IDX64">pregap4: naming schemes</A><LI><A HREF="pregap4_34.html#IDX63">pregap4: Reference sequence</A><LI><A HREF="pregap4_4.html#IDX8">pregap4: temporary files</A><LI><A HREF="pregap4_3.html#IDX2">Pregap4</A><LI><A HREF="pregap4_4.html#IDX5">pregap4rc files</A><LI><A HREF="pregap4_56.html#IDX110">pregap4rc</A><LI><A HREF="trev_8.html#IDX28">Previous button, trev</A><LI><A HREF="gap4_62.html#IDX220">Primer selection: contig editor</A><LI><A HREF="gap4_65.html#IDX298">Primer Selection: contig editor</A><LI><A HREF="gap5_23.html#IDX76">Primer Selection: contig editor</A><LI><A HREF="vector_clip_9.html#IDX10">Primer site, defining</A><LI><A HREF="vector_clip_10.html#IDX17">Primer site, finding</A><LI><A HREF="gap4_180.html#IDX637">Primer types, ignoring</A><LI><A HREF="gap4_109.html#IDX411">Primers: suggestion of</A><LI><A HREF="gap4_62.html#IDX204">Printing contigs</A><LI><A HREF="gap4_62.html#IDX210">printing: aligned readings</A><LI><A HREF="formats_8.html#IDX27">Private data: SCF</A><LI><A HREF="spin_40.html#IDX98">Probabilities in spin</A><LI><A HREF="pregap4_73.html#IDX188">probability</A><LI><A HREF="pregap4_91.html#IDX344">process_dialogue</A><LI><A HREF="spin_40.html#IDX109">protein alignment symbols: spin</A><LI><A HREF="spin_40.html#IDX107">Protein score matrix format</A><LI><A HREF="spin_16.html#IDX20">Protein:spin</A><LI><A HREF="formats_20.html#IDX75">PS: experiment file line type</A></DIR>
<A NAME="LETq"></A><H2>q</H2>
<DIR>
<LI><A HREF="manpages_20.html#IDX21">Qclip: man page</A><LI><A HREF="formats_20.html#IDX76">QL: experiment file line type</A><LI><A HREF="formats_20.html#IDX77">QR: experiment file line type</A><LI><A HREF="gap4_122.html#IDX436">Quality calculation algorithm</A><LI><A HREF="gap5_63.html#IDX174">Quality calculation algorithm</A><LI><A HREF="pregap4_70.html#IDX151">quality clip configuration</A><LI><A HREF="gap4_136.html#IDX470">Quality clip ends</A><LI><A HREF="pregap4_22.html#IDX37">Quality clip module</A><LI><A HREF="gap4_135.html#IDX468">Quality clipping</A><LI><A HREF="gap4_117.html#IDX431">Quality codes</A><LI><A HREF="gap4_63.html#IDX289">Quality in contig editor</A><LI><A HREF="gap4_43.html#IDX99">Quality plot: examining the plot</A><LI><A HREF="gap4_33.html#IDX85">Quality plot: template display</A><LI><A HREF="gap4_43.html#IDX98">Quality plot</A><LI><A HREF="gap4_16.html#IDX23">Quality values - use of</A><LI><A HREF="gap4_63.html#IDX275">Quality values: contig editor, displayed</A><LI><A HREF="gap4_57.html#IDX159">Quality values: contig editor, use within</A><LI><A HREF="gap5_16.html#IDX43">Quality values: contig editor, use within</A><LI><A HREF="gap4_117.html#IDX430">Quality: output for consensus</A><LI><A HREF="pregap4_70.html#IDX152">quality_clip.p4m</A><LI><A HREF="pregap4_88.html#IDX319">quality_clip</A><LI><A HREF="trev_10.html#IDX45">Quit: Trev</A><LI><A HREF="gap4_74.html#IDX326">Quitting: contig editor</A><LI><A HREF="gap5_28.html#IDX97">Quitting: contig editor</A></DIR>
<A NAME="LETr"></A><H2>r</H2>
<DIR>
<LI><A HREF="spin_49.html#IDX167">Range: spin</A><LI><A HREF="gap4_151.html#IDX523">RAWD note type</A><LI><A HREF="gap4_177.html#IDX626">RAWDATA</A><LI><A HREF="gap5_67.html#IDX184">Read groups: SAM RG tags</A><LI><A HREF="gap4_95.html#IDX369">Read pair data and contig ordering</A><LI><A HREF="gap4_96.html#IDX377">Read pairs</A><LI><A HREF="gap5_45.html#IDX141">Read pairs</A><LI><A HREF="gap4_158.html#IDX562">read raid</A><LI><A HREF="spin_49.html#IDX157">Read sequence: spin</A><LI><A HREF="gap4_38.html#IDX91">Read-pair coverage</A><LI><A HREF="gap4_63.html#IDX249">Read-pairs, trace display</A><LI><A HREF="gap4_71.html#IDX315">READ_BRIEF_FORMAT</A><LI><A HREF="gap5_25.html#IDX87">READ_BRIEF_FORMAT</A><LI><A HREF="gap4_181.html#IDX647">read_only (command line option)</A><LI><A HREF="read_clipping_2.html#IDX2">reading clipping</A><LI><A HREF="gap4_37.html#IDX90">Reading coverage</A><LI><A HREF="spin_32.html#IDX78">Reading frame:spin</A><LI><A HREF="gap4_4.html#IDX13">reading length limits in gap4</A><LI><A HREF="gap4_157.html#IDX549">Reading name length restrictions</A><LI><A HREF="gap4_157.html#IDX548">Reading name restrictions</A><LI><A HREF="formats_2.html#IDX1">Reading name restrictions</A><LI><A HREF="gap4_53.html#IDX123">reading names in the editor</A><LI><A HREF="gap5_13.html#IDX38">reading names in the editor</A><LI><A HREF="gap4_145.html#IDX507">Reading names, searching for</A><LI><A HREF="gap5_73.html#IDX213">Reading names, searching for</A><LI><A HREF="gap4_157.html#IDX547">Reading names</A><LI><A HREF="gap4_157.html#IDX555">Reading numbers</A><LI><A HREF="gap4_103.html#IDX403">reading percent mismatch</A><LI><A HREF="gap5_49.html#IDX152">reading percent mismatch</A><LI><A HREF="gap4_30.html#IDX68">Reading plot: template display</A><LI><A HREF="gap4_163.html#IDX576">Reading structure: doctor database</A><LI><A HREF="gap4_32.html#IDX81">Readings list: template display</A><LI><A HREF="gap4_140.html#IDX480">readings list</A><LI><A HREF="gap5_70.html#IDX192">readings list</A><LI><A HREF="gap4_158.html#IDX561">readings: copying to other databases</A><LI><A HREF="gap4_152.html#IDX528">readings: entering</A><LI><A HREF="vector_clip_11.html#IDX25">readings: extraneous</A><LI><A HREF="gap4_157.html#IDX556">Readings: maximum in a database</A><LI><A HREF="gap4_103.html#IDX404">readings: sorted on alignment score</A><LI><A HREF="gap5_49.html#IDX153">readings: sorted on alignment score</A><LI><A HREF="gap4_152.html#IDX535">readonly database</A><LI><A HREF="gap4_152.html#IDX531">readonly</A><LI><A HREF="gap4_158.html#IDX560">reads: copying to other databases</A><LI><A HREF="gap4_128.html#IDX449">realigning sequences</A><LI><A HREF="formats_19.html#IDX41">Records in experiment files</A><LI><A HREF="interface_9.html#IDX22">Redirect output</A><LI><A HREF="gap4_67.html#IDX304">Reference sequence: contig editor</A><LI><A HREF="pregap4_34.html#IDX62">Reference sequence: pregap4</A><LI><A HREF="mutations_4.html#IDX4">reference sequences</A><LI><A HREF="pregap4_34.html#IDX61">Reference trace module</A><LI><A HREF="gap4_67.html#IDX305">Reference traces: contig editor</A><LI><A HREF="mutations_4.html#IDX6">reference traces</A><LI><A HREF="references_1.html#IDX1">references</A><LI><A HREF="trev_8.html#IDX30">Reject button, trev</A><LI><A HREF="gap5_56.html#IDX162">Remove Contig Holes</A><LI><A HREF="gap5_55.html#IDX161">Remove Pad Columns</A><LI><A HREF="gap4_64.html#IDX296">Remove reading: contig editor</A><LI><A HREF="gap4_26.html#IDX59">Remove, in Contig Comparator</A><LI><A HREF="gap5_8.html#IDX27">Remove, in Contig Comparator</A><LI><A HREF="gap5_53.html#IDX159">Removing contigs</A><LI><A HREF="vector_clip_11.html#IDX26">removing extraneous readings</A><LI><A HREF="pregap4_56.html#IDX107">Removing modules</A><LI><A HREF="gap4_106.html#IDX408">Removing readings</A><LI><A HREF="gap5_52.html#IDX157">Removing readings</A><LI><A HREF="gap4_138.html#IDX476">Removing results</A><LI><A HREF="gap5_68.html#IDX188">Removing results</A><LI><A HREF="gap4_101.html#IDX398">Repeat search</A><LI><A HREF="gap5_44.html#IDX139">Repeat search</A><LI><A HREF="pregap4_80.html#IDX228">repeat_file</A><LI><A HREF="pregap4_31.html#IDX56">repeat_masker.p4m</A><LI><A HREF="pregap4_81.html#IDX234">RepeatMasker configuration</A><LI><A HREF="pregap4_31.html#IDX55">RepeatMasker module</A><LI><A HREF="pregap4_81.html#IDX235">repeats_masker.p4m</A><LI><A HREF="gap4_62.html#IDX217">Report mutations: contig editor</A><LI><A HREF="pregap4_91.html#IDX350">report</A><LI><A HREF="formats_23.html#IDX97">Restriction enzyme files</A><LI><A HREF="spin_44.html#IDX139">Restriction enzyme sites:spin</A><LI><A HREF="spin_21.html#IDX27">Restriction enzymes: configuring: spin</A><LI><A HREF="gap4_48.html#IDX110">Restriction enzymes: configuring</A><LI><A HREF="gap5_33.html#IDX106">Restriction enzymes: configuring</A><LI><A HREF="gap4_49.html#IDX114">Restriction enzymes: cut sites</A><LI><A HREF="spin_20.html#IDX25">Restriction enzymes: examining the plot: spin</A><LI><A HREF="gap4_47.html#IDX108">Restriction enzymes: examining the plot</A><LI><A HREF="gap5_32.html#IDX104">Restriction enzymes: examining the plot</A><LI><A HREF="spin_18.html#IDX23">Restriction enzymes: introduction: spin</A><LI><A HREF="spin_19.html#IDX24">Restriction enzymes: selecting enzymes: spin</A><LI><A HREF="gap4_46.html#IDX107">Restriction enzymes: selecting enzymes</A><LI><A HREF="gap5_31.html#IDX103">Restriction enzymes: selecting enzymes</A><LI><A HREF="gap4_49.html#IDX112">Restriction enzymes: tags, creation of</A><LI><A HREF="gap4_34.html#IDX87">Restriction enzymes: template display</A><LI><A HREF="gap4_50.html#IDX115">Restriction enzymes: textual output</A><LI><A HREF="gap5_34.html#IDX107">Restriction enzymes: textual output</A><LI><A HREF="gap4_45.html#IDX106">Restriction enzymes</A><LI><A HREF="gap5_30.html#IDX102">Restriction enzymes</A><LI><A HREF="spin_22.html#IDX29">Restriction site listing:spin</A><LI><A HREF="spin_22.html#IDX28">Restriction site printing:spin</A><LI><A HREF="formats_2.html#IDX7">Restrictions on experiment file names</A><LI><A HREF="formats_2.html#IDX6">Restrictions on file names</A><LI><A HREF="formats_2.html#IDX9">Restrictions on reading names</A><LI><A HREF="formats_2.html#IDX10">Restrictions on sample names</A><LI><A HREF="formats_2.html#IDX8">Restrictions on SCF file names</A><LI><A HREF="gap4_138.html#IDX474">Results manager: introduction</A><LI><A HREF="gap5_68.html#IDX186">Results manager: introduction</A><LI><A HREF="spin_48.html#IDX154">Results manager: spin</A><LI><A HREF="gap4_138.html#IDX473">Results manager</A><LI><A HREF="gap5_68.html#IDX185">Results manager</A><LI><A HREF="gap4_24.html#IDX50">Results menu: Contig Selector</A><LI><A HREF="gap5_6.html#IDX19">Results menu: Contig Selector</A><LI><A HREF="gap4_138.html#IDX475">Results: removing</A><LI><A HREF="gap5_68.html#IDX187">Results: removing</A><LI><A HREF="interface_8.html#IDX17">Right mouse button: overview</A><LI><A HREF="pregap4_70.html#IDX161">right_num_uncalled</A><LI><A HREF="pregap4_70.html#IDX160">right_win_length</A><LI><A HREF="spin_49.html#IDX177">Rotate sequence: spin</A><LI><A HREF="formats_20.html#IDX78">RS: experiment file line type</A><LI><A HREF="gap4_30.html#IDX73">Ruler: template display</A><LI><A HREF="pregap4_12.html#IDX10">Run command</A><LI><A HREF="pregap4_91.html#IDX340">run</A></DIR>
<A NAME="LETs"></A><H2>s</H2>
<DIR>
<LI><A HREF="gap4_157.html#IDX554">sample name restrictions</A><LI><A HREF="formats_2.html#IDX5">Sample name restrictions</A><LI><A HREF="formats_5.html#IDX17">Sample points: SCF</A><LI><A HREF="formats_5.html#IDX19">Samples1: SCF structure</A><LI><A HREF="formats_5.html#IDX20">Samples2: SCF structure</A><LI><A HREF="pregap4_50.html#IDX93">Sanger Centre naming scheme, new</A><LI><A HREF="pregap4_49.html#IDX91">Sanger Centre naming scheme, old</A><LI><A HREF="pregap4_50.html#IDX94">sanger_names_new.p4t</A><LI><A HREF="pregap4_49.html#IDX92">sanger_names_old.p4t</A><LI><A HREF="gap4_156.html#IDX545">Save As</A><LI><A HREF="gap4_62.html#IDX211">Save Consensus Trace: contig editor</A><LI><A HREF="gap4_141.html#IDX492">Save list</A><LI><A HREF="gap5_71.html#IDX204">Save list</A><LI><A HREF="spin_49.html#IDX178">Save sequence: spin</A><LI><A HREF="gap4_62.html#IDX206">saving contigs to file</A><LI><A HREF="gap4_62.html#IDX200">Saving: contig editor</A><LI><A HREF="trev_7.html#IDX26">Saving: Trev</A><LI><A HREF="formats_20.html#IDX79">SC: experiment file line type</A><LI><A HREF="pregap4_71.html#IDX176">SC</A><LI><A HREF="trev_3.html#IDX8">Scaling: Trev</A><LI><A HREF="gap4_157.html#IDX553">SCF file name restrictions</A><LI><A HREF="formats_2.html#IDX3">SCF file name restrictions</A><LI><A HREF="formats_4.html#IDX14">SCF header record</A><LI><A HREF="formats_4.html#IDX16">SCF magic number</A><LI><A HREF="formats_10.html#IDX30">SCF: byte ordering</A><LI><A HREF="formats_7.html#IDX24">SCF: comments</A><LI><A HREF="formats_9.html#IDX28">SCF: file structure</A><LI><A HREF="formats_8.html#IDX26">SCF: private data</A><LI><A HREF="formats_5.html#IDX18">SCF: Sample points</A><LI><A HREF="formats_6.html#IDX21">SCF: sequence</A><LI><A HREF="formats_3.html#IDX11">SCF</A><LI><A HREF="spin_40.html#IDX108">Score matrix format</A><LI><A HREF="pregap4_82.html#IDX251">score</A><LI><A HREF="spin_49.html#IDX176">Scramble sequence: spin</A><LI><A HREF="pregap4_75.html#IDX203">Screen for unclipped vector configuration</A><LI><A HREF="pregap4_26.html#IDX45">Screen for unclipped vector module</A><LI><A HREF="gap4_91.html#IDX364">Screen only: assembly</A><LI><A HREF="pregap4_76.html#IDX210">Screen sequences configuration</A><LI><A HREF="pregap4_27.html#IDX47">Screen sequences module</A><LI><A HREF="pregap4_76.html#IDX215">screen_file</A><LI><A HREF="pregap4_76.html#IDX214">screen_mode</A><LI><A HREF="vector_clip_14.html#IDX27">screen_seq, limits</A><LI><A HREF="pregap4_76.html#IDX211">screen_seq.p4m</A><LI><A HREF="vector_clip_15.html#IDX30">Screen_seq: error codes</A><LI><A HREF="manpages_21.html#IDX23">Screen_seq: error codes</A><LI><A HREF="manpages_21.html#IDX22">screen_seq: man page</A><LI><A HREF="vector_clip_11.html#IDX18">Screen_seq</A><LI><A HREF="vector_clip_11.html#IDX19">screen_seq</A><LI><A HREF="pregap4_75.html#IDX204">screen_vector.p4m</A><LI><A HREF="vector_clip_3.html#IDX2">Screening against vector sequence</A><LI><A HREF="vector_clip_11.html#IDX22">screening for bacterial sequences</A><LI><A HREF="vector_clip_11.html#IDX21">screening for vectors</A><LI><A HREF="vector_clip_11.html#IDX20">Screening readings for contaminant sequences</A><LI><A HREF="interface_9.html#IDX21">Scroll on output</A><LI><A HREF="formats_20.html#IDX80">SE: experiment file line type</A><LI><A HREF="gap4_147.html#IDX513">Search annotation contents: lists</A><LI><A HREF="gap4_130.html#IDX452">Search from file</A><LI><A HREF="gap4_145.html#IDX506">Search sequence names: lists</A><LI><A HREF="gap5_73.html#IDX212">Search sequence names: lists</A><LI><A HREF="gap4_146.html#IDX510">Search template names: lists</A><LI><A HREF="interface_9.html#IDX20">Search: in the output window</A><LI><A HREF="gap4_61.html#IDX178">Searching by annotation comments: contig editor</A><LI><A HREF="gap4_61.html#IDX195">Searching by consensus discrepancies: contig editor</A><LI><A HREF="gap4_61.html#IDX182">Searching by consensus quality: contig editor</A><LI><A HREF="gap4_61.html#IDX193">Searching by discrepancies: contig editor</A><LI><A HREF="gap4_61.html#IDX185">Searching by edits: contig editor</A><LI><A HREF="gap4_61.html#IDX188">Searching by Evidence for Edit1: contig editor</A><LI><A HREF="gap4_61.html#IDX191">Searching by Evidence for Edit2: contig editor</A><LI><A HREF="gap4_61.html#IDX183">Searching by file: contig editor</A><LI><A HREF="gap4_61.html#IDX176">Searching by position: contig editor</A><LI><A HREF="gap4_61.html#IDX177">Searching by problem: contig editor</A><LI><A HREF="gap4_61.html#IDX181">Searching by quality: contig editor</A><LI><A HREF="gap4_61.html#IDX180">Searching by sequence: contig editor</A><LI><A HREF="gap4_61.html#IDX179">Searching by tag type: contig editor</A><LI><A HREF="gap4_61.html#IDX186">Searching by Verify AND: contig editor</A><LI><A HREF="spin_24.html#IDX44">Searching for motifs:spin</A><LI><A HREF="spin_44.html#IDX132">Searching for oligos:spin</A><LI><A HREF="spin_44.html#IDX126">Searching for strings:spin</A><LI><A HREF="gap4_61.html#IDX184">Searching reading name: contig editor</A><LI><A HREF="gap4_61.html#IDX174">Searching: contig editor</A><LI><A HREF="gap5_21.html#IDX61">Searching: contig editor</A><LI><A HREF="spin_25.html#IDX49">Searching: motifs:spin</A><LI><A HREF="spin_27.html#IDX56">Searching: open reading frames:spin</A><LI><A HREF="spin_32.html#IDX80">Searching: protein genes:spin</A><LI><A HREF="spin_32.html#IDX79">Searching: splice sites:spin</A><LI><A HREF="spin_26.html#IDX51">Searching: start codons:spin</A><LI><A HREF="spin_27.html#IDX53">Searching: stop codons:spin</A><LI><A HREF="trev_4.html#IDX12">Searching: Trev</A><LI><A HREF="spin_33.html#IDX81">Searching: tRNA genes:spin</A><LI><A HREF="spin_24.html#IDX43">Searching:spin</A><LI><A HREF="gap4_40.html#IDX94">Second highest confidence graph</A><LI><A HREF="gap4_181.html#IDX654">security</A><LI><A HREF="gap4_31.html#IDX79">Select tags: template display</A><LI><A HREF="spin_49.html#IDX180">Selecting a sequence: spin</A><LI><A HREF="gap4_22.html#IDX40">selecting contigs: Contig Selector</A><LI><A HREF="gap5_4.html#IDX9">selecting contigs: Contig Selector</A><LI><A HREF="gap5_19.html#IDX55">Selecting sequences: contig editor</A><LI><A HREF="gap4_59.html#IDX168">Selections: contig editor</A><LI><A HREF="gap5_18.html#IDX53">Selections: contig editor</A><LI><A HREF="spin_49.html#IDX181">Seq identifier: spin</A><LI><A HREF="spin_13.html#IDX8">Sequence composition:spin</A><LI><A HREF="spin_47.html#IDX153">Sequence display: spin</A><LI><A HREF="spin_43.html#IDX116">Sequence display:spin</A><LI><A HREF="spin_31.html#IDX72">Sequence interpretation: finding protein genes:spin</A><LI><A HREF="spin_33.html#IDX83">Sequence interpretation: tRNA gene search:spin</A><LI><A HREF="gap4_172.html#IDX612">sequence length, maximum</A><LI><A HREF="spin_49.html#IDX164">Sequence manager: spin</A><LI><A HREF="gap4_145.html#IDX508">Sequence names, searching for</A><LI><A HREF="gap5_73.html#IDX214">Sequence names, searching for</A><LI><A HREF="spin_43.html#IDX118">Sequence scrolling:spin</A><LI><A HREF="gap4_131.html#IDX458">Sequence Search</A><LI><A HREF="gap5_47.html#IDX146">Sequence Search</A><LI><A HREF="spin_43.html#IDX117">Sequence viewer:spin</A><LI><A HREF="formats_6.html#IDX22">Sequence: SCF</A><LI><A HREF="pregap4_71.html#IDX165">sequence_vector_clip.p4m</A><LI><A HREF="pregap4_71.html#IDX164">Sequencing vector clip configuration</A><LI><A HREF="pregap4_23.html#IDX39">Sequencing vector clip module</A><LI><A HREF="gap4_63.html#IDX282">Set Active Tags: contig editor</A><LI><A HREF="gap4_63.html#IDX286">Set Default Confidences: contig editor</A><LI><A HREF="spin_15.html#IDX16">Set genetic code:spin</A><LI><A HREF="gap4_175.html#IDX617">Set genetic code</A><LI><A HREF="gap4_63.html#IDX290">Set or unset saving of undo: contig editor</A><LI><A HREF="gap4_63.html#IDX284">Set Output List: contig editor</A><LI><A HREF="spin_49.html#IDX166">Set the range: spin</A><LI><A HREF="gap4_63.html#IDX229">Settings menu: contig editor</A><LI><A HREF="gap5_22.html#IDX63">Settings menu: contig editor</A><LI><A HREF="gap4_63.html#IDX233">Settings: saving in contig editor</A><LI><A HREF="gap5_22.html#IDX67">Settings: saving in contig editor</A><LI><A HREF="formats_20.html#IDX81">SF: experiment file line type</A><LI><A HREF="pregap4_75.html#IDX207">SF</A><LI><A HREF="gap4_177.html#IDX628">SFF= RAWDATA accessor</A><LI><A HREF="gap4_167.html#IDX597">Shift readings: doctor database</A><LI><A HREF="gap4_85.html#IDX353">Shotgun assembly</A><LI><A HREF="gap4_63.html#IDX272">Show consensus quality: contig editor</A><LI><A HREF="gap4_63.html#IDX276">Show Edits: contig editor</A><LI><A HREF="gap4_30.html#IDX71">Show only read pairs: template display</A><LI><A HREF="gap4_63.html#IDX248">Show read-pair Traces</A><LI><A HREF="gap4_63.html#IDX271">Show reading quality: contig editor</A><LI><A HREF="gap4_129.html#IDX451">Show relationships</A><LI><A HREF="gap4_63.html#IDX239">Show Strands: contig editor</A><LI><A HREF="gap4_63.html#IDX280">Show template names</A><LI><A HREF="gap4_63.html#IDX278">Show unpadded positions</A><LI><A HREF="gap4_128.html#IDX450">shuffle pads</A><LI><A HREF="gap5_54.html#IDX160">Shuffle Pads</A><LI><A HREF="pregap4_90.html#IDX328">Shutdown configuration</A><LI><A HREF="pregap4_90.html#IDX329">shutdown.p4m</A><LI><A HREF="pregap4_91.html#IDX341">shutdown</A><LI><A HREF="formats_20.html#IDX82">SI: experiment file line type</A><LI><A HREF="spin_40.html#IDX99">Significance of matches in spin</A><LI><A HREF="spin_39.html#IDX95">Sim: spin</A><LI><A HREF="spin_49.html#IDX161">Simple search: spin</A><LI><A HREF="pregap4_54.html#IDX100">Simple Text Database</A><LI><A HREF="pregap4_81.html#IDX238">simple_only</A><LI><A HREF="gap4_4.html#IDX20">simultaneous database access</A><LI><A HREF="gap4_87.html#IDX356">Single stranded regions: assembling into</A><LI><A HREF="spin_44.html#IDX145">Single stranded sequence listing:spin</A><LI><A HREF="spin_44.html#IDX140">Sites: restriction enzymes:spin</A><LI><A HREF="formats_20.html#IDX83">SL: experiment file line type</A><LI><A HREF="spin_39.html#IDX96">Smith-Waterman: spin</A><LI><A HREF="gap4_42.html#IDX96">SNP candidates</A><LI><A HREF="gap4_27.html#IDX63">Sort Matches</A><LI><A HREF="gap5_9.html#IDX31">Sort Matches</A><LI><A HREF="formats_20.html#IDX84">SP: experiment file line type</A><LI><A HREF="pregap4_71.html#IDX174">SP</A><LI><A HREF="spin_45.html#IDX146">SPIN Sequence Comparison Plot: spin</A><LI><A HREF="spin_42.html#IDX110">SPIN Sequence Plot: spin</A><LI><A HREF="spin_49.html#IDX169">Spin: copy sequence</A><LI><A HREF="spin_49.html#IDX170">Spin: sequence type (linear or circular)</A><LI><A HREF="spin_2.html#IDX1">Spin</A><LI><A HREF="spin_32.html#IDX74">Splice junctions:spin</A><LI><A HREF="spin_32.html#IDX77">Splice sites:spin</A><LI><A HREF="gap4_132.html#IDX461">Splitting databases</A><LI><A HREF="formats_20.html#IDX85">SQ: experiment file line type</A><LI><A HREF="formats_20.html#IDX86">SR: experiment file line type</A><LI><A HREF="formats_20.html#IDX87">SS: experiment file line type</A><LI><A HREF="formats_20.html#IDX88">ST: experiment file line type</A><LI><A HREF="spin_26.html#IDX52">Start codons:spin</A><LI><A HREF="pregap4_82.html#IDX252">start</A><LI><A HREF="gap4_71.html#IDX311">Status line: contig editor</A><LI><A HREF="gap5_25.html#IDX83">Status line: contig editor</A><LI><A HREF="spin_33.html#IDX88">Stems and loops:spin</A><LI><A HREF="gap4_44.html#IDX100">Stop codons display</A><LI><A HREF="gap4_44.html#IDX103">Stop codons: examining the plot</A><LI><A HREF="gap4_44.html#IDX105">Stop codons: updating the plot</A><LI><A HREF="spin_30.html#IDX68">Stop codons:spin</A><LI><A HREF="gap4_111.html#IDX415">Stops: suggested experiments</A><LI><A HREF="gap4_39.html#IDX92">strand coverage</A><LI><A HREF="spin_44.html#IDX130">String finding:spin</A><LI><A HREF="spin_44.html#IDX127">String matching:spin</A><LI><A HREF="gap4_131.html#IDX459">string search</A><LI><A HREF="gap5_47.html#IDX147">string search</A><LI><A HREF="spin_44.html#IDX124">String searching:spin</A><LI><A HREF="pregap4_13.html#IDX19">Styles of windows</A><LI><A HREF="spin_44.html#IDX125">Subsequence searching:spin</A><LI><A HREF="gap4_110.html#IDX412">Suggest long readings</A><LI><A HREF="gap4_109.html#IDX410">Suggest primers</A><LI><A HREF="gap4_112.html#IDX416">Suggest probes</A><LI><A HREF="gap4_58.html#IDX166">Summary of editing commands: contig editor</A><LI><A HREF="gap5_17.html#IDX51">Summary of editing commands: contig editor</A><LI><A HREF="gap4_83.html#IDX337">Summary: contig editor</A><LI><A HREF="gap5_29.html#IDX99">Summary: contig editor</A><LI><A HREF="gap4_95.html#IDX375">Super contigs</A><LI><A HREF="gap4_56.html#IDX130">Superedit: contig editor</A><LI><A HREF="formats_20.html#IDX89">SV: experiment file line type</A></DIR>
<A NAME="LETt"></A><H2>t</H2>
<DIR>
<LI><A HREF="gap4_179.html#IDX634">Tag database</A><LI><A HREF="pregap4_80.html#IDX226">Tag repeats configuration</A><LI><A HREF="pregap4_32.html#IDX57">Tag repeats module</A><LI><A HREF="gap4_178.html#IDX631">Tag Selector</A><LI><A HREF="gap4_71.html#IDX319">TAG_BRIEF_FORMAT</A><LI><A HREF="gap5_25.html#IDX93">TAG_BRIEF_FORMAT</A><LI><A HREF="pregap4_80.html#IDX227">tag_repeats.p4m</A><LI><A HREF="pregap4_81.html#IDX236">tag_type</A><LI><A HREF="pregap4_80.html#IDX230">tag_types</A><LI><A HREF="gap4_147.html#IDX515">Tags, searching for</A><LI><A HREF="gap4_60.html#IDX170">Tags: contig editor</A><LI><A HREF="gap5_20.html#IDX57">Tags: contig editor</A><LI><A HREF="gap4_127.html#IDX444">tags: entering from a file</A><LI><A HREF="gap4_49.html#IDX111">Tags: restriction enzymes plot</A><LI><A HREF="gap4_31.html#IDX80">Tags: template display</A><LI><A HREF="gap4_18.html#IDX31">Tags</A><LI><A HREF="gap4_177.html#IDX627">TAR= RAWDATA accessor</A><LI><A HREF="formats_20.html#IDX91">TC: experiment file line type</A><LI><A HREF="gap4_75.html#IDX328">Techniques of editing</A><LI><A HREF="gap4_95.html#IDX370">Template display and contig ordering</A><LI><A HREF="gap4_32.html#IDX83">Template display: active readings</A><LI><A HREF="gap4_30.html#IDX76">Template display: contig</A><LI><A HREF="gap4_30.html#IDX70">Template display: ignore single templates</A><LI><A HREF="gap4_33.html#IDX84">Template display: quality plot</A><LI><A HREF="gap4_30.html#IDX65">Template Display: reading plot</A><LI><A HREF="gap4_32.html#IDX82">Template display: readings list</A><LI><A HREF="gap4_34.html#IDX86">Template display: restriction enzymes</A><LI><A HREF="gap4_30.html#IDX75">Template display: ruler</A><LI><A HREF="gap4_31.html#IDX78">Template display: select tags</A><LI><A HREF="gap4_30.html#IDX72">Template display: show only read pairs</A><LI><A HREF="gap4_31.html#IDX77">Template display: tags</A><LI><A HREF="gap4_30.html#IDX66">Template Display: template plot</A><LI><A HREF="gap4_29.html#IDX64">Template Display</A><LI><A HREF="gap5_10.html#IDX32">Template Display</A><LI><A HREF="gap4_146.html#IDX511">Template names, searching for</A><LI><A HREF="gap4_30.html#IDX67">Template plot: template display</A><LI><A HREF="gap4_180.html#IDX636">Template size tolerance</A><LI><A HREF="gap4_70.html#IDX308">Template Status Codes</A><LI><A HREF="gap4_180.html#IDX635">Template Status</A><LI><A HREF="gap4_163.html#IDX582">Template structure: doctor database</A><LI><A HREF="gap4_98.html#IDX382">Template: find read pairs</A><LI><A HREF="gap4_63.html#IDX256">Templates: grouping readings in contig editor</A><LI><A HREF="pregap4_4.html#IDX7">temporary files</A><LI><A HREF="interface_6.html#IDX7">Text windows</A><LI><A HREF="formats_20.html#IDX90">TG: experiment file line type</A><LI><A HREF="gap5_36.html#IDX113">tg_index</A><LI><A HREF="gap4_92.html#IDX365">Tips on assembly</A><LI><A HREF="gap4_170.html#IDX608">tk_utilsrc</A><LI><A HREF="formats_20.html#IDX92">TN: experiment file line type</A><LI><A HREF="pregap4_62.html#IDX125">to_scf.p4m</A><LI><A HREF="gap4_63.html#IDX268">Toggle auto-save: contig editor</A><LI><A HREF="pregap4_35.html#IDX67">Trace difference module</A><LI><A HREF="gap4_63.html#IDX250">Trace differences: contig editor</A><LI><A HREF="gap4_63.html#IDX254">Trace differences: Y scaling</A><LI><A HREF="gap4_63.html#IDX245">Trace Display menu: contig editor</A><LI><A HREF="gap4_66.html#IDX302">Trace displays: contig editor</A><LI><A HREF="gap5_24.html#IDX80">Trace displays: contig editor</A><LI><A HREF="gap4_177.html#IDX625">Trace file location</A><LI><A HREF="gap4_4.html#IDX17">trace files: defining location</A><LI><A HREF="gap4_4.html#IDX16">trace files: location</A><LI><A HREF="pregap4_66.html#IDX135">Trace Format Conversion configuration</A><LI><A HREF="pregap4_19.html#IDX30">Trace Format Conversion</A><LI><A HREF="gap4_62.html#IDX214">Trace: Consensus Trace</A><LI><A HREF="pregap4_82.html#IDX250">trace_diff.p4m</A><LI><A HREF="manpages_23.html#IDX25">trace_dump: man page</A><LI><A HREF="manpages_22.html#IDX24">tracediff: man page</A><LI><A HREF="spin_49.html#IDX173">Transcribe sequence: spin</A><LI><A HREF="spin_49.html#IDX175">Translate sequence: spin</A><LI><A HREF="spin_44.html#IDX143">Translation to protein:spin</A><LI><A HREF="gap4_63.html#IDX243">Translations: contig editor</A><LI><A HREF="trev_6.html#IDX21">Trev: editing the sequence</A><LI><A HREF="trev_6.html#IDX15">Trev: editing</A><LI><A HREF="trev_3.html#IDX9">Trev: fonts</A><LI><A HREF="trev_5.html#IDX13">Trev: information</A><LI><A HREF="trev_1.html#IDX1">Trev: introduction</A><LI><A HREF="trev_2.html#IDX3">Trev: opening trace files</A><LI><A HREF="trev_9.html#IDX33">Trev: page options</A><LI><A HREF="trev_9.html#IDX34">Trev: paper options</A><LI><A HREF="trev_9.html#IDX36">Trev: print fonts</A><LI><A HREF="trev_9.html#IDX35">Trev: print panels</A><LI><A HREF="trev_9.html#IDX32">Trev: printing a trace</A><LI><A HREF="trev_10.html#IDX44">Trev: quit</A><LI><A HREF="trev_7.html#IDX25">Trev: saving a trace file</A><LI><A HREF="trev_3.html#IDX7">Trev: scaling</A><LI><A HREF="trev_4.html#IDX11">Trev: searching</A><LI><A HREF="trev_6.html#IDX17">Trev: setting cutoffs</A><LI><A HREF="trev_9.html#IDX41">Trev: trace print bases</A><LI><A HREF="trev_9.html#IDX39">Trev: trace print colour and line width</A><LI><A HREF="trev_9.html#IDX40">Trev: trace print dash pattern</A><LI><A HREF="trev_9.html#IDX43">Trev: trace print example</A><LI><A HREF="trev_9.html#IDX42">Trev: trace print magnification</A><LI><A HREF="trev_9.html#IDX37">Trev: trace print options</A><LI><A HREF="trev_9.html#IDX38">Trev: trace print title</A><LI><A HREF="trev_6.html#IDX23">Trev: undo</A><LI><A HREF="trev_6.html#IDX20">Trev: vector sequence</A><LI><A HREF="trev_1.html#IDX2">Trev</A><LI><A HREF="spin_33.html#IDX82">tRNA gene search:spin</A><LI><A HREF="spin_33.html#IDX87">tRNA introns:spin</A></DIR>
<A NAME="LETu"></A><H2>u</H2>
<DIR>
<LI><A HREF="gap4_144.html#IDX502">Unattached readings: lists</A><LI><A HREF="pregap4_69.html#IDX142">Uncalled base clip configuration</A><LI><A HREF="pregap4_69.html#IDX143">uncalled_clip.p4m</A><LI><A HREF="trev_6.html#IDX24">Undo clip edits, trev</A><LI><A HREF="gap4_63.html#IDX293">Undo toggle: contig editor</A><LI><A HREF="gap4_63.html#IDX292">Undo: contig editor</A><LI><A HREF="spin_31.html#IDX73">Uneven positional base frequencies:spin</A><LI><A HREF="gap4_116.html#IDX428">Unfinished consensus</A><LI><A HREF="gap4_71.html#IDX313">Unpadded base positions</A><LI><A HREF="gap5_25.html#IDX85">Unpadded base positions</A><LI><A HREF="gap4_63.html#IDX279">Unpadded positions in editor</A><LI><A HREF="gap4_95.html#IDX371">Update contig order</A><LI><A HREF="pregap4_82.html#IDX256">update_exp_file</A><LI><A HREF="pregap4_74.html#IDX199">update_expfile</A><LI><A HREF="gap4_177.html#IDX630">URL= RAWDATA accessor</A><LI><A HREF="pregap4_61.html#IDX123">use_sample_name</A><LI><A HREF="pregap4_71.html#IDX170">use_vp_file</A><LI><A HREF="interface_4.html#IDX4">User interface: buttons</A><LI><A HREF="interface_11.html#IDX28">User interface: colour selector</A><LI><A HREF="interface_7.html#IDX10">User interface: entry boxes</A><LI><A HREF="interface_2.html#IDX2">User interface: introduction</A><LI><A HREF="interface_5.html#IDX6">User interface: menus</A><LI><A HREF="interface_6.html#IDX8">User interface: text windows</A><LI><A HREF="interface_10.html#IDX26">User interface: Zooming graphics</A><LI><A HREF="interface_2.html#IDX1">User interface</A><LI><A HREF="gap4_174.html#IDX616">User levels</A></DIR>
<A NAME="LETv"></A><H2>v</H2>
<DIR>
<LI><A HREF="pregap4_91.html#IDX352">Variables, global to all modules</A><LI><A HREF="pregap4_91.html#IDX347">Variables, in modules</A><LI><A HREF="vector_clip_9.html#IDX13">vector file formats</A><LI><A HREF="vector_clip_3.html#IDX3">Vector sequence: screening</A><LI><A HREF="formats_25.html#IDX101">vector sequences format</A><LI><A HREF="vector_clip_9.html#IDX11">Vector_Clip: cloning site, defining</A><LI><A HREF="vector_clip_6.html#IDX6">Vector_Clip: error codes</A><LI><A HREF="manpages_24.html#IDX27">Vector_Clip: error codes</A><LI><A HREF="manpages_24.html#IDX26">vector_clip: man page</A><LI><A HREF="vector_clip_9.html#IDX12">Vector_Clip: primer site, defining</A><LI><A HREF="vector_clip_2.html#IDX1">Vector_Clip</A><LI><A HREF="pregap4_72.html#IDX183">vector_file</A><LI><A HREF="pregap4_71.html#IDX172">vector_list</A><LI><A HREF="vector_clip_8.html#IDX7">Vector_primer files</A><LI><A HREF="formats_24.html#IDX98">Vector_Primer files</A><LI><A HREF="trev_6.html#IDX19">Vectors, in Trev</A><LI><A HREF="gap4_61.html#IDX187">Verfiy AND: contig editor</A><LI><A HREF="gap4_61.html#IDX190">Verfiy OR: contig editor</A><LI><A HREF="gap4_24.html#IDX49">View menu: Contig Selector</A><LI><A HREF="gap5_6.html#IDX18">View menu: Contig Selector</A><LI><A HREF="pregap4_71.html#IDX171">vp_file</A><LI><A HREF="pregap4_71.html#IDX173">vp_length</A></DIR>
<A NAME="LETw"></A><H2>w</H2>
<DIR>
<LI><A HREF="spin_32.html#IDX76">Weight matrix: splice sites:spin</A><LI><A HREF="spin_24.html#IDX42">Weight matrix:spin</A><LI><A HREF="pregap4_13.html#IDX18">Window styles</A><LI><A HREF="pregap4_70.html#IDX153">window_length</A><LI><A HREF="pregap4_74.html#IDX196">word_length</A><LI><A HREF="formats_20.html#IDX93">WT: wild type trace file</A><LI><A HREF="pregap4_82.html#IDX257">WT</A></DIR>
<A NAME="LETy"></A><H2>y</H2>
<DIR>
<LI><A HREF="gap4_63.html#IDX253">Y scale differences</A></DIR>
<A NAME="LETz"></A><H2>z</H2>
<DIR>
<LI><A HREF="spin_46.html#IDX150">Zoom: spin</A><LI><A HREF="interface_10.html#IDX25">Zooming graphics</A><LI><A HREF="formats_14.html#IDX35">ZTR Chunk format</A><LI><A HREF="formats_15.html#IDX36">ZTR Chunk types</A><LI><A HREF="formats_13.html#IDX33">ZTR header</A><LI><A HREF="formats_17.html#IDX38">ZTR References</A><LI><A HREF="formats_16.html#IDX37">ZTR Text Identifiers</A><LI><A HREF="formats_12.html#IDX32">ZTR</A></DIR>
<hr size=4>
<a href="../index.html"><img src="i/nav_home.gif" alt="home"></a>
<a href="master_brief.html"><img src="i/nav_brief.gif" alt="brief"></a>
<a href="master_contents.html"><img src="i/nav_full.gif" alt="full"></a>
<HR>
This document was generated using the <CITE>merge_indexes.pl</CITE> program.
<p>
<i>Last generated on 25 April 2016.
</i>
<font size="-1"><br>
</BODY>
</HTML>
