
<HTML>
<HEAD>
<!-- This HTML file has been created by texi2html 1.39
     from spin.htmlinfo on 25 April 2016 -->

<TITLE>Spin - SPIN-Gene-Finding</TITLE>
</HEAD>
<BODY bgcolor="#ffffff">
<A HREF="spin_1.html"><IMG SRC="i/nav_first.gif" ALT="first"></A>  <A HREF="spin_24.html"><IMG SRC="i/nav_prev.gif" ALT="previous"></A>  <A HREF="spin_26.html"><IMG SRC="i/nav_next.gif" ALT="next"></A>  <A HREF="spin_50.html"><IMG SRC="i/nav_last.gif" ALT="last"></A>  <A HREF="spin_toc.html"><IMG SRC="i/nav_top.gif" ALT="contents"></A>
<HR>
<!-- NODE:SPIN-Gene-Finding -->


<H2><A NAME="SEC35" HREF="spin_toc.html#SEC35">Gene finding</A></H2>
<P>
<A NAME="IDX45"></A>
<A NAME="IDX46"></A>
<A NAME="IDX47"></A>
<A NAME="IDX48"></A>
<A NAME="IDX49"></A>
<A NAME="IDX50"></A>

</P>

<UL>
<LI><A HREF="spin_26.html#SEC36">Start Codon Search</A>
<LI><A HREF="spin_27.html#SEC37">Stop Codon Search</A>
<LI><A HREF="spin_28.html#SEC38">Codon Usage Method</A>
<LI><A HREF="spin_30.html#SEC40">Author Test</A>
<LI><A HREF="spin_31.html#SEC41">Uneven Positional Base Frequencies Method</A>
<LI><A HREF="spin_32.html#SEC42">Splice Site Search</A>
<LI><A HREF="spin_24.html#SEC34">Motif Search</A>
<LI><A HREF="spin_33.html#SEC43">tRNA Gene Search</A>
</UL>

<P>
Many years ago <CITE>Staden R. (1984) Graphic methods to determine
the function of nucleic acid sequences. Nucl. Acids Res. 12, 521-538</CITE>
we separated methods for searching for genes and their
control regions into two classes: "gene search by signal", and "gene
search by content". 
<CITE>Staden R. (1985) Computer methods to locate genes  and  signals  in
nucleic acid sequences, Genetic Engineering: Principles
and Methods Vol. 7, Edited  by  J. K. Setlow   and   A.
Hollaender, Plenum Publishing Corp.</CITE>.
Signal searches look for short segments of
sequences such as promoters, ribosome binding sites, splice junctions,
etc, whereas content searches look for the sequence patterns that are
characteristic of protein coding regions, or RNA genes. Protein coding
sequences produce particular amino acid sequences, often using preferred
codons, and this leaves patterns in the sequence that can be used to
distinguish them from non-protein-coding DNA. tRNA genes must produce
stable cloverleaf structures and "standard" tRNAs must contain
particular (conserved) bases at locations within the cloverleaf. These
features can be used to locate tRNA genes, and probably other RNA
genes could be sought in a similar way.

</P>
<P>
The methods described in the following sections are either "content" or
"signal" searches and spin's graphical presentation of results can be used to
see if together they produce a consistent gene prediction. 

</P>
<HR>
<A HREF="spin_1.html"><IMG SRC="i/nav_first.gif" ALT="first"></A>  <A HREF="spin_24.html"><IMG SRC="i/nav_prev.gif" ALT="previous"></A>  <A HREF="spin_26.html"><IMG SRC="i/nav_next.gif" ALT="next"></A>  <A HREF="spin_50.html"><IMG SRC="i/nav_last.gif" ALT="last"></A>  <A HREF="spin_toc.html"><IMG SRC="i/nav_top.gif" ALT="contents"></A>
<hr>
<i>Last generated on 25 April 2016.</i>
<font size="-1"><br>
</font>
</BODY>
</HTML>
